{
 "cells": [
  {
   "cell_type": "code",
   "id": "initial_id",
   "metadata": {
    "collapsed": true,
    "ExecuteTime": {
     "end_time": "2025-10-12T11:34:58.215996Z",
     "start_time": "2025-10-12T11:34:56.867791Z"
    }
   },
   "source": "import pandas as pd",
   "outputs": [],
   "execution_count": 1
  },
  {
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-10-12T11:35:10.416227Z",
     "start_time": "2025-10-12T11:35:10.390353Z"
    }
   },
   "cell_type": "code",
   "source": "df = pd.read_csv('../data/raw/SB_publication_PMC.csv', encoding='utf-8')",
   "id": "6a250fb2358a677",
   "outputs": [],
   "execution_count": 2
  },
  {
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-10-12T11:35:13.681563Z",
     "start_time": "2025-10-12T11:35:13.651780Z"
    }
   },
   "cell_type": "code",
   "source": "df.head()",
   "id": "b41a391e4347f7e6",
   "outputs": [
    {
     "data": {
      "text/plain": [
       "                                               Title  \\\n",
       "0  Mice in Bion-M 1 space mission: training and s...   \n",
       "1  Microgravity induces pelvic bone loss through ...   \n",
       "2  Stem Cell Health and Tissue Regeneration in Mi...   \n",
       "3  Microgravity Reduces the Differentiation and R...   \n",
       "4  Microgravity validation of a novel system for ...   \n",
       "\n",
       "                                                Link  \n",
       "0  https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4...  \n",
       "1  https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3...  \n",
       "2  https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1...  \n",
       "3  https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7...  \n",
       "4  https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5...  "
      ],
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Title</th>\n",
       "      <th>Link</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>Mice in Bion-M 1 space mission: training and s...</td>\n",
       "      <td>https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>Microgravity induces pelvic bone loss through ...</td>\n",
       "      <td>https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>Stem Cell Health and Tissue Regeneration in Mi...</td>\n",
       "      <td>https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>Microgravity Reduces the Differentiation and R...</td>\n",
       "      <td>https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>Microgravity validation of a novel system for ...</td>\n",
       "      <td>https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5...</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ]
     },
     "execution_count": 3,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "execution_count": 3
  },
  {
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-10-12T11:36:42.921939Z",
     "start_time": "2025-10-12T11:36:42.907833Z"
    }
   },
   "cell_type": "code",
   "source": "df['Link'].tolist()",
   "id": "69ac1e4ac77e2233",
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4136787/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3630201/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11988870/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7998608/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5587110/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8396460/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5666799/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5460236/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6222041/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6813909/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4095884/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3040128/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3177255/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11500582/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5387210/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4642138/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5387210/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2915878/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3901686/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6985101/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6387434/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6371294/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7072278/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8441986/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9400218/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9267413/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9576569/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10789781/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10772081/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11166946/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11166944/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11166968/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7000411/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7787258/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8716943/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4826010/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6048781/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7010715/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7503278/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8364238/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11579474/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2998437/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3005423/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3190158/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3289768/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3508904/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3430326/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3593973/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5018776/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4896697/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11127935/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11831363/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11930778/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8816950/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3774184/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4118556/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4653813/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6915713/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6124165/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8509868/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11063234/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5672023/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5899691/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6204554/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4064004/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3818365/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3981873/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4150462/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4378170/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4618186/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4453782/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4902601/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4776492/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5415411/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6289879/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7987364/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8099722/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5116466/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4033213/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6081456/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5955502/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10285634/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11271499/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6062551/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9832585/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7829349/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3570223/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3457586/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3558598/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11046949/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3890248/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3868799/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4960141/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5736159/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5761896/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5826609/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6366624/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11929063/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11929063/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10996920/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7954810/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4228280/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3639165/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8044432/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7778922/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7870178/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7733874/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7828077/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8044432/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11166911/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11167097/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11094041/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11053165/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11403809/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8513672/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11126634/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11386075/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3502426/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3856860/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3615599/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4050424/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3659353/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4169763/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4398884/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4379453/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4385880/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4964660/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5866446/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6165321/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7339929/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8220224/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2910419/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3166430/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6615562/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7324008/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8412175/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10607959/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9699585/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9953055/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10528075/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10813126/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10390562/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4290804/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5515520/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5286820/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5470433/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5491145/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5996828/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6201722/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6240453/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6447593/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7264257/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7064724/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6379395/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4187166/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5219934/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6386654/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7940393/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9023564/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9389742/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2824534/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2991213/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8269219/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11362537/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9111996/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10308117/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10233975/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10848226/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3329368/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC12008199/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC12008199/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4035928/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC12008199/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5181587/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4936552/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7610290/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5614317/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5677518/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11850895/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7076552/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7414185/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8133610/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8113475/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9706465/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10027818/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3337602/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4490751/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5132293/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11747068/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5666834/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6753329/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7012842/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4405755/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8260663/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4647464/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5659752/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6372189/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6945029/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7000411/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8191917/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9865768/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10797188/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10797188/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7555395/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4309212/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6560652/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6321533/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC12040010/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11833055/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9146534/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11492218/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6339989/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4110898/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3947616/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5477841/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9139491/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5515531/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3258067/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10025027/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7733933/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9105288/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9617909/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9695138/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3503204/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5896955/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4469364/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4411250/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5047659/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5454470/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5515506/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3044105/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3002170/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6188462/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11470607/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3603133/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6462350/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4012541/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4653058/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6137544/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2913424/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7190111/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7382458/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8444978/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9241228/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8953707/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11166646/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3747666/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4337661/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4631774/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5052530/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5114340/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5488280/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9293686/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6091969/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7029147/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3092937/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5748516/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9287483/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5748516/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4211383/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5748516/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5821596/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6889863/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6908503/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7607443/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10764921/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9605285/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3962621/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4462657/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5026643/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5026642/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10342025/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11167039/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10020673/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9144607/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7314936/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4340379/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5933128/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8185232/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3936501/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11998595/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7996555/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8234954/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4152162/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5498037/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6547757/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6747492/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6746706/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8400407/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9365836/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9547011/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10138634/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC12031868/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9583032/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9746133/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10422843/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11022651/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11094041/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11357997/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11167063/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11166909/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11222149/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11166948/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11166952/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11166937/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11357981/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11166969/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11166943/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11166967/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11167060/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11166981/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11263583/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11484870/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10780891/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7067979/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4321547/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6710492/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC12008199/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8518694/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9861679/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2897429/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4187989/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8217198/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6981245/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6981245/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6131065/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11233762/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6213004/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7171750/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10414970/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8450295/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8183356/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8932657/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10504369/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3982735/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4682325/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10264680/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7358545/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9108708/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7555797/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3545801/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3869332/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4135744/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3911009/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4923109/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11673879/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5648920/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5817088/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7825584/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8904044/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3012082/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11088941/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3187170/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3133089/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3242767/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3067220/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3251573/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3647661/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4085587/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4096993/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7467030/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4427280/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5460263/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6204695/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7943786/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11888247/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3851762/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8067245/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8044432/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9701605/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11166648/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11166662/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3838407/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3748764/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2925951/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3350920/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3422896/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11869762/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10712242/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6143729/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10058394/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4103452/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8024390/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6348315/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6359015/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7667275/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7756144/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8169688/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8475764/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9098553/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10503492/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9693084/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3792163/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4430214/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5045078/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5443495/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5666744/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5865701/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6337482/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9953463/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10487739/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11942576/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4052902/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8539686/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10800490/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7756143/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3066201/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11324864/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3665236/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10470837/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10386755/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8896891/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8430797/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5460135/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10774393/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6126739/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4411111/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6126739/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9445043/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3509447/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5600826/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10831389/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10831389/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4507784/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4349987/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6638264/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6130020/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9695374/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141014/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6706399/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC12034070/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7235020/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7599661/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8493566/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10926278/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6069985/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9856674/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9410522/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10344948/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4750446/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7162976/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11353732/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11953390/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9737020/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5515519/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10144393/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10719374/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11096397/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11166655/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7937622/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7055474/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7118179/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4494396/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8323809/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8886715/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8890767/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9487896/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10142442/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11477029/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4309301/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11166981/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6343501/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7138367/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6689164/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3564669/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4042434/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4161659/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4917201/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4985531/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5421618/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5515514/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8671758/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5771473/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5288125/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5389344/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9784906/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6548323/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6597714/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6865368/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7907224/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8213760/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9582271/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10063413/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10889206/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8468678/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9031868/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10030976/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5910807/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC12021249/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6057834/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6876156/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7415877/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7285199/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9549344/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7530743/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7890410/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9640571/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9862956/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10410709/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10751425/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10605753/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10846184/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11166967/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11302229/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11487192/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3554398/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10370681/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4624184/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4945788/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9116463/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5201052/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5391430/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5504618/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5580210/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8421509/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5552977/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5805884/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5864415/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6013642/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5578834/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9241228/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5958250/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7326050/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6251167/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6280456/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9098231/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9116463/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6426649/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6452512/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6529585/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6952069/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7303416/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7317102/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7393961/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7484075/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7516158/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7561690/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7595942/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7677455/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8005752/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8104057/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC12034939/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8012687/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8086211/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8211661/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11988970/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8754149/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8875396/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9258157/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9503880/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9743659/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10233975/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10308117/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10715203/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10628120/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10960378/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10628120/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11451251/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5523082/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6590338/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6036641/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6599637/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8274610/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8131644/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9163032/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5568470/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5485990/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5460239/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6275019/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5397022/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7251690/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10284894/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10030976/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8421509/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9964234/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10793275/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4116675/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5810432/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10261121/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8739323/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8739323/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9502502/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10432549/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7261848/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8537831/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8879990/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10472590/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11999716/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11969330/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11940681/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11941215/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11892206/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11748630/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11659408/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11701653/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11593819/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11477029/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11487192/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11484870/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11451251/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11403809/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11386075/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11362537/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11339457/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11324864/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11353732/',\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11271499/']"
      ]
     },
     "execution_count": 6,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "execution_count": 6
  },
  {
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-10-12T11:42:54.794696Z",
     "start_time": "2025-10-12T11:39:04.788689Z"
    }
   },
   "cell_type": "code",
   "source": [
    "from docling.document_converter import DocumentConverter\n",
    "\n",
    "source = \"https://arxiv.org/pdf/2408.09869\"  # file path or URL\n",
    "converter = DocumentConverter()\n",
    "doc = converter.convert(source).document\n",
    "print(doc.export_to_markdown())"
   ],
   "id": "6a2115cbca2337f9",
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "C:\\Users\\compu\\anaconda3\\envs\\env-conda\\Lib\\site-packages\\tqdm\\auto.py:21: TqdmWarning: IProgress not found. Please update jupyter and ipywidgets. See https://ipywidgets.readthedocs.io/en/stable/user_install.html\n",
      "  from .autonotebook import tqdm as notebook_tqdm\n",
      "2025-10-12 08:39:22,370 - INFO - detected formats: [<InputFormat.PDF: 'pdf'>]\n",
      "2025-10-12 08:39:22,554 - INFO - Going to convert document batch...\n",
      "2025-10-12 08:39:22,563 - INFO - Initializing pipeline for StandardPdfPipeline with options hash e647edf348883bed75367b22fbe60347\n",
      "2025-10-12 08:39:22,631 - INFO - Loading plugin 'docling_defaults'\n",
      "2025-10-12 08:39:22,635 - INFO - Registered picture descriptions: ['vlm', 'api']\n",
      "2025-10-12 08:39:22,687 - INFO - Loading plugin 'docling_defaults'\n",
      "2025-10-12 08:39:22,708 - INFO - Registered ocr engines: ['easyocr', 'ocrmac', 'rapidocr', 'tesserocr', 'tesseract']\n",
      "2025-10-12 08:39:22,921 - INFO - Accelerator device: 'cpu'\n",
      "2025-10-12 08:39:22,931 - WARNING - Downloading detection model, please wait. This may take several minutes depending upon your network connection.\n",
      "2025-10-12 08:39:41,058 - INFO - Download complete\n",
      "2025-10-12 08:39:41,060 - WARNING - Downloading recognition model, please wait. This may take several minutes depending upon your network connection.\n",
      "2025-10-12 08:39:45,329 - INFO - Download complete.\n",
      "2025-10-12 08:39:48,089 - INFO - Accelerator device: 'cpu'\n",
      "Xet Storage is enabled for this repo, but the 'hf_xet' package is not installed. Falling back to regular HTTP download. For better performance, install the package with: `pip install huggingface_hub[hf_xet]` or `pip install hf_xet`\n",
      "2025-10-12 08:39:49,704 - WARNING - Xet Storage is enabled for this repo, but the 'hf_xet' package is not installed. Falling back to regular HTTP download. For better performance, install the package with: `pip install huggingface_hub[hf_xet]` or `pip install hf_xet`\n",
      "C:\\Users\\compu\\anaconda3\\envs\\env-conda\\Lib\\site-packages\\huggingface_hub\\file_download.py:143: UserWarning: `huggingface_hub` cache-system uses symlinks by default to efficiently store duplicated files but your machine does not support them in C:\\Users\\compu\\.cache\\huggingface\\hub\\models--ds4sd--docling-layout-heron. Caching files will still work but in a degraded version that might require more space on your disk. This warning can be disabled by setting the `HF_HUB_DISABLE_SYMLINKS_WARNING` environment variable. For more details, see https://huggingface.co/docs/huggingface_hub/how-to-cache#limitations.\n",
      "To support symlinks on Windows, you either need to activate Developer Mode or to run Python as an administrator. In order to activate developer mode, see this article: https://docs.microsoft.com/en-us/windows/apps/get-started/enable-your-device-for-development\n",
      "  warnings.warn(message)\n",
      "Xet Storage is enabled for this repo, but the 'hf_xet' package is not installed. Falling back to regular HTTP download. For better performance, install the package with: `pip install huggingface_hub[hf_xet]` or `pip install hf_xet`\n",
      "2025-10-12 08:40:37,323 - WARNING - Xet Storage is enabled for this repo, but the 'hf_xet' package is not installed. Falling back to regular HTTP download. For better performance, install the package with: `pip install huggingface_hub[hf_xet]` or `pip install hf_xet`\n",
      "Xet Storage is enabled for this repo, but the 'hf_xet' package is not installed. Falling back to regular HTTP download. For better performance, install the package with: `pip install huggingface_hub[hf_xet]` or `pip install hf_xet`\n",
      "2025-10-12 08:40:37,334 - WARNING - Xet Storage is enabled for this repo, but the 'hf_xet' package is not installed. Falling back to regular HTTP download. For better performance, install the package with: `pip install huggingface_hub[hf_xet]` or `pip install hf_xet`\n",
      "C:\\Users\\compu\\anaconda3\\envs\\env-conda\\Lib\\site-packages\\huggingface_hub\\file_download.py:143: UserWarning: `huggingface_hub` cache-system uses symlinks by default to efficiently store duplicated files but your machine does not support them in C:\\Users\\compu\\.cache\\huggingface\\hub\\models--ds4sd--docling-models. Caching files will still work but in a degraded version that might require more space on your disk. This warning can be disabled by setting the `HF_HUB_DISABLE_SYMLINKS_WARNING` environment variable. For more details, see https://huggingface.co/docs/huggingface_hub/how-to-cache#limitations.\n",
      "To support symlinks on Windows, you either need to activate Developer Mode or to run Python as an administrator. In order to activate developer mode, see this article: https://docs.microsoft.com/en-us/windows/apps/get-started/enable-your-device-for-development\n",
      "  warnings.warn(message)\n",
      "2025-10-12 08:42:02,415 - INFO - Accelerator device: 'cpu'\n",
      "2025-10-12 08:42:02,891 - INFO - Processing document 2408.09869v5.pdf\n",
      "C:\\Users\\compu\\anaconda3\\envs\\env-conda\\Lib\\site-packages\\torch\\utils\\data\\dataloader.py:666: UserWarning: 'pin_memory' argument is set as true but no accelerator is found, then device pinned memory won't be used.\n",
      "  warnings.warn(warn_msg)\n",
      "C:\\Users\\compu\\anaconda3\\envs\\env-conda\\Lib\\site-packages\\torch\\utils\\data\\dataloader.py:666: UserWarning: 'pin_memory' argument is set as true but no accelerator is found, then device pinned memory won't be used.\n",
      "  warnings.warn(warn_msg)\n",
      "2025-10-12 08:42:54,699 - INFO - Finished converting document 2408.09869v5.pdf in 214.23 sec.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "<!-- image -->\n",
      "\n",
      "## Docling Technical Report\n",
      "\n",
      "## Version 1.0\n",
      "\n",
      "Christoph Auer Maksym Lysak Ahmed Nassar Michele Dolfi Nikolaos Livathinos Panos Vagenas Cesar Berrospi Ramis Matteo Omenetti Fabian Lindlbauer Kasper Dinkla Lokesh Mishra Yusik Kim Shubham Gupta Rafael Teixeira de Lima Valery Weber Lucas Morin Ingmar Meijer Viktor Kuropiatnyk Peter W. J. Staar\n",
      "\n",
      "AI4K Group, IBM Research R uschlikon, Switzerland\n",
      "\n",
      "## Abstract\n",
      "\n",
      "This technical report introduces Docling , an easy to use, self-contained, MITlicensed open-source package for PDF document conversion. It is powered by state-of-the-art specialized AI models for layout analysis (DocLayNet) and table structure recognition (TableFormer), and runs efficiently on commodity hardware in a small resource budget. The code interface allows for easy extensibility and addition of new features and models.\n",
      "\n",
      "## 1 Introduction\n",
      "\n",
      "Converting PDF documents back into a machine-processable format has been a major challenge for decades due to their huge variability in formats, weak standardization and printing-optimized characteristic, which discards most structural features and metadata. With the advent of LLMs and popular application patterns such as retrieval-augmented generation (RAG), leveraging the rich content embedded in PDFs has become ever more relevant. In the past decade, several powerful document understanding solutions have emerged on the market, most of which are commercial software, cloud offerings [3] and most recently, multi-modal vision-language models. As of today, only a handful of open-source tools cover PDF conversion, leaving a significant feature and quality gap to proprietary solutions.\n",
      "\n",
      "With Docling , we open-source a very capable and efficient document conversion tool which builds on the powerful, specialized AI models and datasets for layout analysis and table structure recognition we developed and presented in the recent past [12, 13, 9]. Docling is designed as a simple, self-contained python library with permissive license, running entirely locally on commodity hardware. Its code architecture allows for easy extensibility and addition of new features and models.\n",
      "\n",
      "Here is what Docling delivers today:\n",
      "\n",
      "- Converts PDF documents to JSON or Markdown format, stable and lightning fast\n",
      "- Understands detailed page layout, reading order, locates figures and recovers table structures\n",
      "- Extracts metadata from the document, such as title, authors, references and language\n",
      "- Optionally applies OCR, e.g. for scanned PDFs\n",
      "- Can be configured to be optimal for batch-mode (i.e high throughput, low time-to-solution) or interactive mode (compromise on efficiency, low time-to-solution)\n",
      "- Can leverage different accelerators (GPU, MPS, etc).\n",
      "\n",
      "## 2 Getting Started\n",
      "\n",
      "To use Docling, you can simply install the docling package from PyPI. Documentation and examples are available in our GitHub repository at github.com/DS4SD/docling. All required model assets 1 are downloaded to a local huggingface datasets cache on first use, unless you choose to pre-install the model assets in advance.\n",
      "\n",
      "Docling provides an easy code interface to convert PDF documents from file system, URLs or binary streams, and retrieve the output in either JSON or Markdown format. For convenience, separate methods are offered to convert single documents or batches of documents. A basic usage example is illustrated below. Further examples are available in the Doclign code repository.\n",
      "\n",
      "from docling.document\\_converter import DocumentConverter\n",
      "\n",
      "```\n",
      "source = \"https://arxiv.org/pdf/2206.01062\" # PDF path or URL converter = DocumentConverter() result = converter.convert_single(source) print(result.render_as_markdown()) # output: \"## DocLayNet: A Large Human -Annotated Dataset for Document -Layout Analysis [...]\"\n",
      "```\n",
      "\n",
      "Optionally, you can configure custom pipeline features and runtime options, such as turning on or off features (e.g. OCR, table structure recognition), enforcing limits on the input document size, and defining the budget of CPU threads. Advanced usage examples and options are documented in the README file. Docling also provides a Dockerfile to demonstrate how to install and run it inside a container.\n",
      "\n",
      "## 3 Processing pipeline\n",
      "\n",
      "Docling implements a linear pipeline of operations, which execute sequentially on each given document (see Fig. 1). Each document is first parsed by a PDF backend, which retrieves the programmatic text tokens, consisting of string content and its coordinates on the page, and also renders a bitmap image of each page to support downstream operations. Then, the standard model pipeline applies a sequence of AI models independently on every page in the document to extract features and content, such as layout and table structures. Finally, the results from all pages are aggregated and passed through a post-processing stage, which augments metadata, detects the document language, infers reading-order and eventually assembles a typed document object which can be serialized to JSON or Markdown.\n",
      "\n",
      "## 3.1 PDF backends\n",
      "\n",
      "Two basic requirements to process PDF documents in our pipeline are a) to retrieve all text content and their geometric coordinates on each page and b) to render the visual representation of each page as it would appear in a PDF viewer. Both these requirements are encapsulated in Docling's PDF backend interface. While there are several open-source PDF parsing libraries available for python, we faced major obstacles with all of them for different reasons, among which were restrictive\n",
      "\n",
      "1 see huggingface.co/ds4sd/docling-models/\n",
      "\n",
      "Figure 1: Sketch of Docling's default processing pipeline. The inner part of the model pipeline is easily customizable and extensible.\n",
      "\n",
      "<!-- image -->\n",
      "\n",
      "licensing (e.g. pymupdf [7]), poor speed or unrecoverable quality issues, such as merged text cells across far-apart text tokens or table columns (pypdfium, PyPDF) [15, 14].\n",
      "\n",
      "We therefore decided to provide multiple backend choices, and additionally open-source a custombuilt PDF parser, which is based on the low-level qpdf [4] library. It is made available in a separate package named docling-parse and powers the default PDF backend in Docling. As an alternative, we provide a PDF backend relying on pypdfium , which may be a safe backup choice in certain cases, e.g. if issues are seen with particular font encodings.\n",
      "\n",
      "## 3.2 AI models\n",
      "\n",
      "As part of Docling, we initially release two highly capable AI models to the open-source community, which have been developed and published recently by our team. The first model is a layout analysis model, an accurate object-detector for page elements [13]. The second model is TableFormer [12, 9], a state-of-the-art table structure recognition model. We provide the pre-trained weights (hosted on huggingface) and a separate package for the inference code as docling-ibm-models . Both models are also powering the open-access deepsearch-experience, our cloud-native service for knowledge exploration tasks.\n",
      "\n",
      "## Layout Analysis Model\n",
      "\n",
      "Our layout analysis model is an object-detector which predicts the bounding-boxes and classes of various elements on the image of a given page. Its architecture is derived from RT-DETR [16] and re-trained on DocLayNet [13], our popular human-annotated dataset for document-layout analysis, among other proprietary datasets. For inference, our implementation relies on the onnxruntime [5].\n",
      "\n",
      "The Docling pipeline feeds page images at 72 dpi resolution, which can be processed on a single CPU with sub-second latency. All predicted bounding-box proposals for document elements are post-processed to remove overlapping proposals based on confidence and size, and then intersected with the text tokens in the PDF to group them into meaningful and complete units such as paragraphs, section titles, list items, captions, figures or tables.\n",
      "\n",
      "## Table Structure Recognition\n",
      "\n",
      "The TableFormer model [12], first published in 2022 and since refined with a custom structure token language [9], is a vision-transformer model for table structure recovery. It can predict the logical row and column structure of a given table based on an input image, and determine which table cells belong to column headers, row headers or the table body. Compared to earlier approaches, TableFormer handles many characteristics of tables, such as partial or no borderlines, empty cells, rows or columns, cell spans and hierarchy both on column-heading or row-heading level, tables with inconsistent indentation or alignment and other complexities. For inference, our implementation relies on PyTorch [2].\n",
      "\n",
      "The Docling pipeline feeds all table objects detected in the layout analysis to the TableFormer model, by providing an image-crop of the table and the included text cells. TableFormer structure predictions are matched back to the PDF cells in post-processing to avoid expensive re-transcription text in the table image. Typical tables require between 2 and 6 seconds to be processed on a standard CPU, strongly depending on the amount of included table cells.\n",
      "\n",
      "## OCR\n",
      "\n",
      "Docling provides optional support for OCR, for example to cover scanned PDFs or content in bitmaps images embedded on a page. In our initial release, we rely on EasyOCR [1], a popular thirdparty OCR library with support for many languages. Docling, by default, feeds a high-resolution page image (216 dpi) to the OCR engine, to allow capturing small print detail in decent quality. While EasyOCR delivers reasonable transcription quality, we observe that it runs fairly slow on CPU (upwards of 30 seconds per page).\n",
      "\n",
      "We are actively seeking collaboration from the open-source community to extend Docling with additional OCR backends and speed improvements.\n",
      "\n",
      "## 3.3 Assembly\n",
      "\n",
      "In the final pipeline stage, Docling assembles all prediction results produced on each page into a well-defined datatype that encapsulates a converted document, as defined in the auxiliary package docling-core . The generated document object is passed through a post-processing model which leverages several algorithms to augment features, such as detection of the document language, correcting the reading order, matching figures with captions and labelling metadata such as title, authors and references. The final output can then be serialized to JSON or transformed into a Markdown representation at the users request.\n",
      "\n",
      "## 3.4 Extensibility\n",
      "\n",
      "Docling provides a straight-forward interface to extend its capabilities, namely the model pipeline. A model pipeline constitutes the central part in the processing, following initial document parsing and preceding output assembly, and can be fully customized by sub-classing from an abstract baseclass ( BaseModelPipeline ) or cloning the default model pipeline. This effectively allows to fully customize the chain of models, add or replace models, and introduce additional pipeline configuration parameters. To use a custom model pipeline, the custom pipeline class to instantiate can be provided as an argument to the main document conversion methods. We invite everyone in the community to propose additional or alternative models and improvements.\n",
      "\n",
      "Implementations of model classes must satisfy the python Callable interface. The \\_\\_call\\_\\_ method must accept an iterator over page objects, and produce another iterator over the page objects which were augmented with the additional features predicted by the model, by extending the provided PagePredictions data model accordingly.\n",
      "\n",
      "## 4 Performance\n",
      "\n",
      "In this section, we establish some reference numbers for the processing speed of Docling and the resource budget it requires. All tests in this section are run with default options on our standard test set distributed with Docling, which consists of three papers from arXiv and two IBM Redbooks, with a total of 225 pages. Measurements were taken using both available PDF backends on two different hardware systems: one MacBook Pro M3 Max, and one bare-metal server running Ubuntu 20.04 LTS on an Intel Xeon E5-2690 CPU. For reproducibility, we fixed the thread budget (through setting OMP NUM THREADS environment variable ) once to 4 (Docling default) and once to 16 (equal to full core count on the test hardware). All results are shown in Table 1.\n",
      "\n",
      "If you need to run Docling in very low-resource environments, please consider configuring the pypdfium backend. While it is faster and more memory efficient than the default docling-parse backend, it will come at the expense of worse quality results, especially in table structure recovery.\n",
      "\n",
      "Establishing GPU acceleration support for the AI models is currently work-in-progress and largely untested, but may work implicitly when CUDA is available and discovered by the onnxruntime and\n",
      "\n",
      "torch runtimes backing the Docling pipeline. We will deliver updates on this topic at in a future version of this report.\n",
      "\n",
      "Table 1: Runtime characteristics of Docling with the standard model pipeline and settings, on our test dataset of 225 pages, on two different systems. OCR is disabled. We show the time-to-solution (TTS), computed throughput in pages per second, and the peak memory used (resident set size) for both the Docling-native PDF backend and for the pypdfium backend, using 4 and 16 threads.\n",
      "\n",
      "| CPU                     | Thread budget   | native backend   | native backend   | native backend   | pypdfium backend   | pypdfium backend   | pypdfium backend   |\n",
      "|-------------------------|-----------------|------------------|------------------|------------------|--------------------|--------------------|--------------------|\n",
      "|                         |                 | TTS              | Pages/s          | Mem              | TTS                | Pages/s            | Mem                |\n",
      "| Apple M3 Max (16 cores) | 4 16            | 177 s 167 s      | 1.27 1.34        | 6.20 GB          | 103 s 92 s         | 2.18 2.45          | 2.56 GB            |\n",
      "| Intel(R) Xeon E5-2690   | 4 16            | 375 s 244 s      | 0.60 0.92        | 6.16 GB          | 239 s 143 s        | 0.94 1.57          | 2.42 GB            |\n",
      "\n",
      "## 5 Applications\n",
      "\n",
      "Thanks to the high-quality, richly structured document conversion achieved by Docling, its output qualifies for numerous downstream applications. For example, Docling can provide a base for detailed enterprise document search, passage retrieval or classification use-cases, or support knowledge extraction pipelines, allowing specific treatment of different structures in the document, such as tables, figures, section structure or references. For popular generative AI application patterns, such as retrieval-augmented generation (RAG), we provide quackling , an open-source package which capitalizes on Docling's feature-rich document output to enable document-native optimized vector embedding and chunking. It plugs in seamlessly with LLM frameworks such as LlamaIndex [8]. Since Docling is fast, stable and cheap to run, it also makes for an excellent choice to build document-derived datasets. With its powerful table structure recognition, it provides significant benefit to automated knowledge-base construction [11, 10]. Docling is also integrated within the open IBM data prep kit [6], which implements scalable data transforms to build large-scale multi-modal training datasets.\n",
      "\n",
      "## 6 Future work and contributions\n",
      "\n",
      "Docling is designed to allow easy extension of the model library and pipelines. In the future, we plan to extend Docling with several more models, such as a figure-classifier model, an equationrecognition model, a code-recognition model and more. This will help improve the quality of conversion for specific types of content, as well as augment extracted document metadata with additional information. Further investment into testing and optimizing GPU acceleration as well as improving the Docling-native PDF backend are on our roadmap, too.\n",
      "\n",
      "We encourage everyone to propose or implement additional features and models, and will gladly take your inputs and contributions under review . The codebase of Docling is open for use and contribution, under the MIT license agreement and in alignment with our contributing guidelines included in the Docling repository. If you use Docling in your projects, please consider citing this technical report.\n",
      "\n",
      "## References\n",
      "\n",
      "- [1] J. AI. Easyocr: Ready-to-use ocr with 80+ supported languages. https://github.com/ JaidedAI/EasyOCR , 2024. Version: 1.7.0.\n",
      "- [2] J. Ansel, E. Yang, H. He, N. Gimelshein, A. Jain, M. Voznesensky, B. Bao, P. Bell, D. Berard, E. Burovski, G. Chauhan, A. Chourdia, W. Constable, A. Desmaison, Z. DeVito, E. Ellison, W. Feng, J. Gong, M. Gschwind, B. Hirsh, S. Huang, K. Kalambarkar, L. Kirsch, M. Lazos, M. Lezcano, Y. Liang, J. Liang, Y. Lu, C. Luk, B. Maher, Y. Pan, C. Puhrsch, M. Reso, M. Saroufim, M. Y. Siraichi, H. Suk, M. Suo, P. Tillet, E. Wang, X. Wang, W. Wen, S. Zhang, X. Zhao, K. Zhou, R. Zou, A. Mathews, G. Chanan, P. Wu, and S. Chintala. Pytorch 2: Faster\n",
      "\n",
      "machine learning through dynamic python bytecode transformation and graph compilation. In Proceedings of the 29th ACM International Conference on Architectural Support for Programming Languages and Operating Systems, Volume 2 (ASPLOS '24) . ACM, 4 2024. doi: 10.1145/3620665.3640366. URL https://pytorch.org/assets/pytorch2-2.pdf .\n",
      "\n",
      "- [3] C. Auer, M. Dolfi, A. Carvalho, C. B. Ramis, and P. W. Staar. Delivering document conversion as a cloud service with high throughput and responsiveness. In 2022 IEEE 15th International Conference on Cloud Computing (CLOUD) , pages 363-373. IEEE, 2022.\n",
      "- [4] J. Berkenbilt. Qpdf: A content-preserving pdf document transformer, 2024. URL https: //github.com/qpdf/qpdf .\n",
      "- [5] O. R. developers. Onnx runtime. https://onnxruntime.ai/ , 2024. Version: 1.18.1.\n",
      "- [6] IBM. Data Prep Kit: a community project to democratize and accelerate unstructured data preparation for LLM app developers, 2024. URL https://github.com/IBM/ data-prep-kit .\n",
      "- [7] A. S. Inc. PyMuPDF, 2024. URL https://github.com/pymupdf/PyMuPDF .\n",
      "- [8] J. Liu. LlamaIndex, 11 2022. URL https://github.com/jerryjliu/llama\\_index .\n",
      "- [9] M. Lysak, A. Nassar, N. Livathinos, C. Auer, and P. Staar. Optimized Table Tokenization for Table Structure Recognition. In Document Analysis and Recognition - ICDAR 2023: 17th International Conference, San Jos e, CA, USA, August 21-26, 2023, Proceedings, Part II , pages 37-50, Berlin, Heidelberg, Aug. 2023. Springer-Verlag. ISBN 978-3-031-41678-1. doi: 10. 1007/978-3-031-41679-8 3. URL https://doi.org/10.1007/978-3-031-41679-8\\_3 .\n",
      "- [10] L. Mishra, S. Dhibi, Y. Kim, C. Berrospi Ramis, S. Gupta, M. Dolfi, and P. Staar. Statements: Universal information extraction from tables with large language models for ESG KPIs. In D. Stammbach, J. Ni, T. Schimanski, K. Dutia, A. Singh, J. Bingler, C. Christiaen, N. Kushwaha, V. Muccione, S. A. Vaghefi, and M. Leippold, editors, Proceedings of the 1st Workshop on Natural Language Processing Meets Climate Change (ClimateNLP 2024) , pages 193-214, Bangkok, Thailand, Aug. 2024. Association for Computational Linguistics. URL https://aclanthology.org/2024.climatenlp-1.15 .\n",
      "- [11] L. Morin, V. Weber, G. I. Meijer, F. Yu, and P. W. J. Staar. Patcid: an open-access dataset of chemical structures in patent documents. Nature Communications , 15(1):6532, August 2024. ISSN 2041-1723. doi: 10.1038/s41467-024-50779-y. URL https://doi.org/10.1038/ s41467-024-50779-y .\n",
      "- [12] A. Nassar, N. Livathinos, M. Lysak, and P. Staar. Tableformer: Table structure understanding with transformers. In Proceedings of the IEEE/CVF Conference on Computer Vision and Pattern Recognition , pages 4614-4623, 2022.\n",
      "- [13] B. Pfitzmann, C. Auer, M. Dolfi, A. S. Nassar, and P. Staar. Doclaynet: a large humanannotated dataset for document-layout segmentation. pages 3743-3751, 2022.\n",
      "- [14] pypdf Maintainers. pypdf: A Pure-Python PDF Library, 2024. URL https://github.com/ py-pdf/pypdf .\n",
      "- [15] P. Team. PyPDFium2: Python bindings for PDFium, 2024. URL https://github.com/ pypdfium2-team/pypdfium2 .\n",
      "- [16] Y. Zhao, W. Lv, S. Xu, J. Wei, G. Wang, Q. Dang, Y. Liu, and J. Chen. Detrs beat yolos on real-time object detection, 2023.\n",
      "\n",
      "## Appendix\n",
      "\n",
      "In this section, we illustrate a few examples of Docling's output in Markdown and JSON.\n",
      "\n",
      "<!-- image -->\n",
      "\n",
      "1 INTRODUCTION\n",
      "\n",
      "Despite the substantial improvements achieved with machine-learning (ML) approaches and deep neural networks in recent years, document conversion remains a challenging problem, as demonstrated by the numerous public competitions held on this topic [1-4]. The challenge originates from the huge variability in PDF documents regarding layout, language and formats (scanned, programmatic or a combination of both). Engineering a single ML model that can be applied on all types of documents and provides high-quality layout segmentation remains to this day extremely challenging [5]. To highlight the variability in document layouts, we show a few example documents from the DocLayNet dataset in Figure 1. Figure 2: Title page of the DocLayNet paper (arxiv.org/pdf/2206.01062) - left PDF, right rendered Markdown. If recognized, metadata such as authors are appearing first under the title. Text content inside figures is currently dropped, the caption is retained and linked to the figure in the JSON representation (not shown).\n",
      "\n",
      "KDD '22, August 14-18, 2022, Washington, DC, USA Birgit Pfitzmann, Christoph Auer, Michele Dolfi, Ahmed S. Nassar, and Peter Staar\n",
      "\n",
      "Table 2: Prediction performance (mAP@0.5-0.95) of object detection networks on DocLayNet test set. The MRCNN (Mask R-CNN) and FRCNN (Faster R-CNN) models with ResNet-50 or ResNet-101 backbone were trained based on the network architectures from the detectron2 model zoo (Mask R-CNN R50, R101-FPN 3x, Faster R-CNN R101-FPN 3x), with default configurations. The YOLO implementation utilized was YOLOv5x6 [13]. All models were initialised using pre-trained weights from the COCO 2017 dataset.\n",
      "\n",
      "|                                                                                                        | human                                                                   | MRCNN R50 R101                                                                                                          | FRCNN R101                                                  | YOLO v5x6                                                   |\n",
      "|--------------------------------------------------------------------------------------------------------|-------------------------------------------------------------------------|-------------------------------------------------------------------------------------------------------------------------|-------------------------------------------------------------|-------------------------------------------------------------|\n",
      "| Caption Footnote Formula List-item Page-footer Page-header Picture Section-header Table Text Title All | 84-89 83-91 83-85 87-88 93-94 85-89 69-71 83-84 77-81 84-86 60-72 82-83 | 68.4 71.5 70.9 71.8 60.1 63.4 81.2 80.8 61.6 59.3 71.9 70.0 71.7 72.7 67.6 69.3 82.2 82.9 84.6 85.8 76.7 80.4 72.4 73.5 | 70.1 73.7 63.5 81.0 58.9 72.0 72.0 68.4 82.2 85.4 79.9 73.4 | 77.7 77.2 66.2 86.2 61.1 67.9 77.1 74.6 86.3 88.1 82.7 76.8 |\n",
      "\n",
      "to avoid this at any cost in order to have clear, unbiased baseline numbers for human document-layout annotation. Third, we introduced the feature of snapping boxes around text segments to obtain a pixel-accurate annotation and again reduce time and effort. The CCS annotation tool automatically shrinks every user-drawn box to the minimum bounding-box around the enclosed text-cells for all purely text-based segments, which excludes only Table and Picture . For the latter, we instructed annotation staff to minimise inclusion of surrounding whitespace while including all graphical lines. A downside of snapping boxes to enclosed text cells is that some wrongly parsed PDF pages cannot be annotated correctly and need to be skipped. Fourth, we established a way to flag pages as rejected for cases where no valid annotation according to the label guidelines could be achieved. Example cases for this would be PDF pages that render incorrectly or contain layouts that are impossible to capture with non-overlapping rectangles. Such rejected pages are not contained in the final dataset. With all these measures in place, experienced annotation staff managed to annotate a single page in a typical timeframe of 20s to 60s, depending on its complexity.\n",
      "\n",
      "## 5 EXPERIMENTS\n",
      "\n",
      "The primary goal of DocLayNet is to obtain high-quality ML models capable of accurate document-layout analysis on a wide variety of challenging layouts. As discussed in Section 2, object detection models are currently the easiest to use, due to the standardisation of ground-truth data in COCO format [16] and the availability of general frameworks such as detectron2 [17]. Furthermore, baseline numbers in PubLayNet and DocBank were obtained using standard object detection models such as Mask R-CNN and Faster R-CNN. As such, we will relate to these object detection methods in this\n",
      "\n",
      "Figure 5: Prediction performance (mAP@0.5-0.95) of a Mask R-CNNnetworkwithResNet50backbonetrainedonincreasing fractions of the DocLayNet dataset. The learning curve flattens around the 80% mark, indicating that increasing the size of the DocLayNet dataset with similar data will not yield significantly better predictions.\n",
      "\n",
      "<!-- image -->\n",
      "\n",
      "paper and leave the detailed evaluation of more recent methods mentioned in Section 2 for future work.\n",
      "\n",
      "In this section, we will present several aspects related to the performance of object detection models on DocLayNet. Similarly as in PubLayNet, we will evaluate the quality of their predictions using mean average precision (mAP) with 10 overlaps that range from 0.5 to 0.95 in steps of 0.05 (mAP@0.5-0.95). These scores are computed by leveraging the evaluation code provided by the COCO API [16].\n",
      "\n",
      "## Baselines for Object Detection\n",
      "\n",
      "In Table 2, we present baseline experiments (given in mAP) on Mask R-CNN [12], Faster R-CNN [11], and YOLOv5 [13]. Both training and evaluation were performed on RGB images with dimensions of 1025  1025 pixels. For training, we only used one annotation in case of redundantly annotated pages. As one can observe, the variation in mAP between the models is rather low, but overall between 6 and 10% lower than the mAP computed from the pairwise human annotations on triple-annotated pages. This gives a good indication that the DocLayNet dataset poses a worthwhile challenge for the research community to close the gap between human recognition and ML approaches. It is interesting to see that Mask R-CNN and Faster R-CNN produce very comparable mAP scores, indicating that pixel-based image segmentation derived from bounding-boxes does not help to obtain better predictions. On the other hand, the more recent Yolov5x model does very well and even out-performs humans on selected labels such as Text , Table and Picture . This is not entirely surprising, as Text , Table and Picture are abundant and the most visually distinctive in a document.\n",
      "\n",
      "mak enbrel\n",
      "\n",
      "Figure 3: Page 6 of the DocLayNet paper. If recognized, metadata such as authors are appearing first under the title. Elements recognized as page headers or footers are suppressed in Markdown to deliver uninterrupted content in reading order. Tables are inserted in reading order. The paragraph in '5. Experiments' wrapping over the column end is broken up in two and interrupted by the table.\n",
      "\n",
      "Third, Ccs achienec Exanole\n",
      "\n",
      "## EXPERIMENTS\n",
      "\n",
      "ground-vuth dawa such WC\n",
      "\n",
      "## Baselines for Object Detection\n",
      "\n",
      "KDD '22, August 14-18, 2022, Washington, DC, USA\n",
      "\n",
      "Birgit Pfitzmann, Christoph Auer, Michele Dolfi, Ahmed S. Nassar, and Peter Staar\n",
      "\n",
      "Table 1: DocLayNet dataset overview. Along with the frequency of each class label, we present the relative occurrence (as %\n",
      "\n",
      "<!-- image -->\n",
      "\n",
      "of row 'Total') in the train, test and validation sets. The inter-annotator agreement is computed as the mAP@0.5-0.95 metric we distributed the annotation workload and performed continuous\n",
      "\n",
      "only. For phases three and four, a group of 40 dedicated annotators quality controls. Phase one and two required a small team of experts\n",
      "\n",
      "were assembled and supervised.\n",
      "\n",
      "while coverage ensures that all meaningful items on a page can to a document category, such as\n",
      "\n",
      "be annotated. We refrained from class labels that are very specific\n",
      "\n",
      "Abstract in the\n",
      "\n",
      "Scientific Articles semantics of the text. Labels such as\n",
      "\n",
      "category. We also avoided class labels that are tightly linked to the\n",
      "\n",
      "Author\n",
      "\n",
      "Affiliation\n",
      "\n",
      "teria for documents were described in Section 3. A large effort went into ensuring that all documents are free to use. The data sources in DocBank, are often only distinguishable by discriminating on 3 https://arxiv.org/ Figure 4: Table 1 from the DocLayNet paper in the original PDF (A), as rendered Markdown (B) and in JSON representation (C). Spanning table cells, such as the multi-column header 'triple interannotator mAP@0.5-0.95 (%)', is repeated for each column in the Markdown representation (B), which guarantees that every data point can be traced back to row and column headings only by its grid coordinates in the table. In the JSON representation, the span information is reflected in the fields of each table cell (C).\n",
      "\n",
      "and\n",
      "\n",
      ", as seen\n",
      "\n",
      "Phase 1: Data selection and preparation.\n",
      "\n",
      "Our inclusion cri-\n"
     ]
    }
   ],
   "execution_count": 7
  },
  {
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-10-12T11:45:57.161578Z",
     "start_time": "2025-10-12T11:45:57.093572Z"
    }
   },
   "cell_type": "code",
   "source": [
    "link1 = df['Link'].tolist()[0]\n",
    "link1"
   ],
   "id": "904282fba43fd9d",
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4136787/'"
      ]
     },
     "execution_count": 8,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "execution_count": 8
  },
  {
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-10-12T11:46:07.957553Z",
     "start_time": "2025-10-12T11:46:01.827666Z"
    }
   },
   "cell_type": "code",
   "source": [
    "\n",
    "doc = converter.convert(link1).document\n",
    "print(doc.export_to_markdown())"
   ],
   "id": "854da61782460e1f",
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "2025-10-12 08:46:03,321 - INFO - detected formats: [<InputFormat.HTML: 'html'>]\n",
      "2025-10-12 08:46:03,439 - INFO - Going to convert document batch...\n",
      "2025-10-12 08:46:03,444 - INFO - Initializing pipeline for SimplePipeline with options hash 995a146ad601044538e6a923bea22f4e\n",
      "2025-10-12 08:46:03,451 - INFO - Processing document PMC4136787\n",
      "2025-10-12 08:46:03,517 - INFO - deleted item in tree at stack: (20, 30, 2, 4, 1) => #/texts/187\n",
      "2025-10-12 08:46:03,527 - INFO - deleted item in tree at stack: (20, 30, 2, 4, 1) => #/texts/187\n",
      "2025-10-12 08:46:03,535 - INFO - deleted item in tree at stack: (20, 30, 2, 4, 1) => #/texts/187\n",
      "2025-10-12 08:46:03,547 - INFO - deleted item in tree at stack: (20, 30, 2, 4, 1) => #/texts/187\n",
      "2025-10-12 08:46:03,556 - INFO - deleted item in tree at stack: (20, 30, 2, 4, 1) => #/texts/187\n",
      "2025-10-12 08:46:03,566 - INFO - deleted item in tree at stack: (20, 30, 2, 4, 1) => #/texts/187\n",
      "2025-10-12 08:46:03,576 - INFO - deleted item in tree at stack: (20, 30, 2, 4, 1) => #/texts/187\n",
      "2025-10-12 08:46:03,586 - INFO - deleted item in tree at stack: (20, 30, 2, 4, 1) => #/texts/187\n",
      "2025-10-12 08:46:03,596 - INFO - deleted item in tree at stack: (20, 30, 2, 4, 1) => #/texts/187\n",
      "2025-10-12 08:46:03,606 - INFO - deleted item in tree at stack: (20, 30, 2, 4, 1) => #/texts/187\n",
      "2025-10-12 08:46:03,616 - INFO - deleted item in tree at stack: (20, 30, 2, 4, 1) => #/texts/187\n",
      "2025-10-12 08:46:03,626 - INFO - deleted item in tree at stack: (20, 30, 2, 4, 1) => #/texts/187\n",
      "2025-10-12 08:46:03,636 - INFO - deleted item in tree at stack: (20, 30, 2, 4, 1) => #/texts/187\n",
      "2025-10-12 08:46:03,645 - INFO - deleted item in tree at stack: (20, 30, 2, 4, 1) => #/texts/187\n",
      "2025-10-12 08:46:03,655 - INFO - deleted item in tree at stack: (20, 30, 2, 4, 1) => #/texts/187\n",
      "2025-10-12 08:46:03,664 - INFO - deleted item in tree at stack: (20, 30, 2, 4, 1) => #/texts/187\n",
      "2025-10-12 08:46:03,673 - INFO - deleted item in tree at stack: (20, 30, 2, 4, 1) => #/texts/187\n",
      "2025-10-12 08:46:03,682 - INFO - deleted item in tree at stack: (20, 30, 2, 4, 1) => #/texts/187\n",
      "2025-10-12 08:46:03,690 - INFO - deleted item in tree at stack: (20, 30, 2, 4, 1) => #/texts/187\n",
      "2025-10-12 08:46:03,699 - INFO - deleted item in tree at stack: (20, 30, 2, 4, 1) => #/texts/187\n",
      "2025-10-12 08:46:03,709 - INFO - deleted item in tree at stack: (20, 30, 2, 4, 1) => #/texts/187\n",
      "2025-10-12 08:46:03,718 - INFO - deleted item in tree at stack: (20, 30, 2, 4, 1) => #/texts/187\n",
      "2025-10-12 08:46:03,727 - INFO - deleted item in tree at stack: (20, 30, 2, 4, 1) => #/texts/187\n",
      "2025-10-12 08:46:03,737 - INFO - deleted item in tree at stack: (20, 30, 2, 4, 1) => #/texts/187\n",
      "2025-10-12 08:46:03,747 - INFO - deleted item in tree at stack: (20, 30, 2, 4, 1) => #/texts/187\n",
      "2025-10-12 08:46:03,755 - INFO - deleted item in tree at stack: (20, 30, 2, 4, 1) => #/texts/187\n",
      "2025-10-12 08:46:03,764 - INFO - deleted item in tree at stack: (20, 30, 2, 4, 1) => #/texts/187\n",
      "2025-10-12 08:46:03,774 - INFO - deleted item in tree at stack: (20, 30, 2, 4, 1) => #/texts/187\n",
      "2025-10-12 08:46:03,784 - INFO - deleted item in tree at stack: (20, 30, 2, 4, 1) => #/texts/187\n",
      "2025-10-12 08:46:03,793 - INFO - deleted item in tree at stack: (20, 30, 2, 4, 1) => #/texts/187\n",
      "2025-10-12 08:46:03,802 - INFO - deleted item in tree at stack: (20, 30, 2, 4, 1) => #/texts/187\n",
      "2025-10-12 08:46:03,811 - INFO - deleted item in tree at stack: (20, 30, 2, 4, 1) => #/texts/187\n",
      "2025-10-12 08:46:03,820 - INFO - deleted item in tree at stack: (20, 30, 2, 4, 1) => #/texts/187\n",
      "2025-10-12 08:46:03,829 - INFO - deleted item in tree at stack: (20, 30, 2, 4, 1) => #/texts/187\n",
      "2025-10-12 08:46:03,838 - INFO - deleted item in tree at stack: (20, 30, 2, 4, 1) => #/texts/187\n",
      "2025-10-12 08:46:03,847 - INFO - deleted item in tree at stack: (20, 30, 2, 4, 1) => #/texts/187\n",
      "2025-10-12 08:46:03,855 - INFO - deleted item in tree at stack: (20, 30, 2, 4, 1) => #/texts/187\n",
      "2025-10-12 08:46:03,864 - INFO - deleted item in tree at stack: (20, 30, 2, 4, 1) => #/texts/187\n",
      "2025-10-12 08:46:03,874 - INFO - deleted item in tree at stack: (20, 30, 2, 4, 1) => #/texts/187\n",
      "2025-10-12 08:46:03,884 - INFO - deleted item in tree at stack: (20, 30, 2, 4, 1) => #/texts/187\n",
      "2025-10-12 08:46:03,893 - INFO - deleted item in tree at stack: (20, 30, 2, 4, 1) => #/texts/187\n",
      "2025-10-12 08:46:03,902 - INFO - deleted item in tree at stack: (20, 30, 2, 4, 1) => #/texts/187\n",
      "2025-10-12 08:46:03,909 - INFO - deleted item in tree at stack: (20, 30, 2, 4, 1) => #/texts/187\n",
      "2025-10-12 08:46:03,919 - INFO - deleted item in tree at stack: (20, 30, 2, 4, 1) => #/texts/187\n",
      "2025-10-12 08:46:03,928 - INFO - deleted item in tree at stack: (20, 30, 2, 4, 1) => #/texts/187\n",
      "2025-10-12 08:46:03,937 - INFO - deleted item in tree at stack: (20, 30, 2, 4, 1) => #/texts/187\n",
      "2025-10-12 08:46:03,947 - INFO - deleted item in tree at stack: (20, 30, 2, 4, 1) => #/texts/187\n",
      "2025-10-12 08:46:03,956 - INFO - deleted item in tree at stack: (20, 30, 2, 4, 1) => #/texts/187\n",
      "2025-10-12 08:46:03,966 - INFO - deleted item in tree at stack: (20, 30, 2, 4, 1) => #/texts/187\n",
      "2025-10-12 08:46:03,975 - INFO - deleted item in tree at stack: (20, 30, 2, 4, 1) => #/texts/187\n",
      "2025-10-12 08:46:03,984 - INFO - deleted item in tree at stack: (20, 30, 2, 4, 1) => #/texts/187\n",
      "2025-10-12 08:46:03,995 - INFO - deleted item in tree at stack: (20, 30, 2, 4, 1) => #/texts/187\n",
      "2025-10-12 08:46:04,003 - INFO - deleted item in tree at stack: (20, 30, 2, 4, 1) => #/texts/187\n",
      "2025-10-12 08:46:04,012 - INFO - deleted item in tree at stack: (20, 30, 2, 4, 1) => #/texts/187\n",
      "2025-10-12 08:46:04,021 - INFO - deleted item in tree at stack: (20, 30, 2, 4, 1) => #/texts/187\n",
      "2025-10-12 08:46:04,033 - INFO - deleted item in tree at stack: (20, 30, 2, 4, 1) => #/texts/187\n",
      "2025-10-12 08:46:04,044 - INFO - deleted item in tree at stack: (20, 30, 2, 4, 1) => #/texts/187\n",
      "2025-10-12 08:46:04,053 - INFO - deleted item in tree at stack: (20, 30, 2, 4, 1) => #/texts/187\n",
      "2025-10-12 08:46:04,062 - INFO - deleted item in tree at stack: (20, 30, 2, 4, 1) => #/texts/187\n",
      "2025-10-12 08:46:04,073 - INFO - deleted item in tree at stack: (20, 30, 2, 5, 1) => #/texts/189\n",
      "2025-10-12 08:46:04,082 - INFO - deleted item in tree at stack: (20, 30, 2, 5, 1) => #/texts/189\n",
      "2025-10-12 08:46:04,091 - INFO - deleted item in tree at stack: (20, 30, 2, 5, 1) => #/texts/189\n",
      "2025-10-12 08:46:04,100 - INFO - deleted item in tree at stack: (20, 30, 2, 5, 1) => #/texts/189\n",
      "2025-10-12 08:46:04,110 - INFO - deleted item in tree at stack: (20, 30, 2, 5, 1) => #/texts/189\n",
      "2025-10-12 08:46:04,119 - INFO - deleted item in tree at stack: (20, 30, 2, 5, 1) => #/texts/189\n",
      "2025-10-12 08:46:04,129 - INFO - deleted item in tree at stack: (20, 30, 2, 5, 1) => #/texts/189\n",
      "2025-10-12 08:46:04,138 - INFO - deleted item in tree at stack: (20, 30, 2, 5, 1) => #/texts/189\n",
      "2025-10-12 08:46:04,149 - INFO - deleted item in tree at stack: (20, 30, 2, 5, 1) => #/texts/189\n",
      "2025-10-12 08:46:04,157 - INFO - deleted item in tree at stack: (20, 30, 2, 5, 1) => #/texts/189\n",
      "2025-10-12 08:46:04,167 - INFO - deleted item in tree at stack: (20, 30, 2, 5, 1) => #/texts/189\n",
      "2025-10-12 08:46:04,176 - INFO - deleted item in tree at stack: (20, 30, 2, 5, 1) => #/texts/189\n",
      "2025-10-12 08:46:04,187 - INFO - deleted item in tree at stack: (20, 30, 2, 5, 1) => #/texts/189\n",
      "2025-10-12 08:46:04,196 - INFO - deleted item in tree at stack: (20, 30, 2, 5, 1) => #/texts/189\n",
      "2025-10-12 08:46:04,204 - INFO - deleted item in tree at stack: (20, 30, 2, 5, 1) => #/texts/189\n",
      "2025-10-12 08:46:04,214 - INFO - deleted item in tree at stack: (20, 30, 2, 5, 1) => #/texts/189\n",
      "2025-10-12 08:46:04,223 - INFO - deleted item in tree at stack: (20, 30, 2, 5, 1) => #/texts/189\n",
      "2025-10-12 08:46:04,232 - INFO - deleted item in tree at stack: (20, 30, 2, 5, 1) => #/texts/189\n",
      "2025-10-12 08:46:04,240 - INFO - deleted item in tree at stack: (20, 30, 2, 5, 1) => #/texts/189\n",
      "2025-10-12 08:46:04,250 - INFO - deleted item in tree at stack: (20, 30, 2, 5, 1) => #/texts/189\n",
      "2025-10-12 08:46:04,261 - INFO - deleted item in tree at stack: (20, 30, 2, 5, 1) => #/texts/189\n",
      "2025-10-12 08:46:04,269 - INFO - deleted item in tree at stack: (20, 30, 2, 5, 1) => #/texts/189\n",
      "2025-10-12 08:46:04,279 - INFO - deleted item in tree at stack: (20, 30, 2, 5, 1) => #/texts/189\n",
      "2025-10-12 08:46:04,288 - INFO - deleted item in tree at stack: (20, 30, 2, 5, 1) => #/texts/189\n",
      "2025-10-12 08:46:04,297 - INFO - deleted item in tree at stack: (20, 30, 2, 5, 1) => #/texts/189\n",
      "2025-10-12 08:46:04,307 - INFO - deleted item in tree at stack: (20, 30, 2, 5, 1) => #/texts/189\n",
      "2025-10-12 08:46:04,316 - INFO - deleted item in tree at stack: (20, 30, 2, 5, 1) => #/texts/189\n",
      "2025-10-12 08:46:04,325 - INFO - deleted item in tree at stack: (20, 30, 2, 5, 1) => #/texts/189\n",
      "2025-10-12 08:46:04,335 - INFO - deleted item in tree at stack: (20, 30, 2, 5, 1) => #/texts/189\n",
      "2025-10-12 08:46:04,343 - INFO - deleted item in tree at stack: (20, 30, 2, 5, 1) => #/texts/189\n",
      "2025-10-12 08:46:04,353 - INFO - deleted item in tree at stack: (20, 30, 2, 5, 1) => #/texts/189\n",
      "2025-10-12 08:46:04,363 - INFO - deleted item in tree at stack: (20, 30, 2, 5, 1) => #/texts/189\n",
      "2025-10-12 08:46:04,372 - INFO - deleted item in tree at stack: (20, 30, 2, 5, 1) => #/texts/189\n",
      "2025-10-12 08:46:04,382 - INFO - deleted item in tree at stack: (20, 30, 2, 5, 1) => #/texts/189\n",
      "2025-10-12 08:46:04,392 - INFO - deleted item in tree at stack: (20, 30, 2, 5, 1) => #/texts/189\n",
      "2025-10-12 08:46:04,401 - INFO - deleted item in tree at stack: (20, 30, 2, 5, 1) => #/texts/189\n",
      "2025-10-12 08:46:04,411 - INFO - deleted item in tree at stack: (20, 30, 2, 5, 1) => #/texts/189\n",
      "2025-10-12 08:46:04,420 - INFO - deleted item in tree at stack: (20, 30, 2, 5, 1) => #/texts/189\n",
      "2025-10-12 08:46:04,430 - INFO - deleted item in tree at stack: (20, 30, 2, 5, 1) => #/texts/189\n",
      "2025-10-12 08:46:04,439 - INFO - deleted item in tree at stack: (20, 30, 2, 5, 1) => #/texts/189\n",
      "2025-10-12 08:46:04,450 - INFO - deleted item in tree at stack: (20, 30, 2, 5, 1) => #/texts/189\n",
      "2025-10-12 08:46:04,471 - INFO - deleted item in tree at stack: (20, 31, 3, 1, 1) => #/texts/292\n",
      "2025-10-12 08:46:04,487 - INFO - deleted item in tree at stack: (20, 31, 3, 1, 1) => #/texts/292\n",
      "2025-10-12 08:46:04,500 - INFO - deleted item in tree at stack: (20, 31, 3, 1, 1) => #/texts/292\n",
      "2025-10-12 08:46:04,514 - INFO - deleted item in tree at stack: (20, 31, 3, 1, 1) => #/texts/292\n",
      "2025-10-12 08:46:04,531 - INFO - deleted item in tree at stack: (20, 31, 3, 1, 1) => #/texts/292\n",
      "2025-10-12 08:46:04,546 - INFO - deleted item in tree at stack: (20, 31, 3, 1, 1) => #/texts/292\n",
      "2025-10-12 08:46:04,559 - INFO - deleted item in tree at stack: (20, 31, 3, 1, 1) => #/texts/292\n",
      "2025-10-12 08:46:04,571 - INFO - deleted item in tree at stack: (20, 31, 3, 1, 1) => #/texts/292\n",
      "2025-10-12 08:46:04,589 - INFO - deleted item in tree at stack: (20, 31, 3, 1, 1) => #/texts/292\n",
      "2025-10-12 08:46:04,602 - INFO - deleted item in tree at stack: (20, 31, 3, 1, 1) => #/texts/292\n",
      "2025-10-12 08:46:04,615 - INFO - deleted item in tree at stack: (20, 31, 3, 1, 1) => #/texts/292\n",
      "2025-10-12 08:46:04,628 - INFO - deleted item in tree at stack: (20, 31, 3, 1, 1) => #/texts/292\n",
      "2025-10-12 08:46:04,642 - INFO - deleted item in tree at stack: (20, 31, 3, 1, 1) => #/texts/292\n",
      "2025-10-12 08:46:04,657 - INFO - deleted item in tree at stack: (20, 31, 3, 1, 1) => #/texts/292\n",
      "2025-10-12 08:46:04,670 - INFO - deleted item in tree at stack: (20, 31, 3, 1, 1) => #/texts/292\n",
      "2025-10-12 08:46:04,684 - INFO - deleted item in tree at stack: (20, 31, 3, 1, 1) => #/texts/292\n",
      "2025-10-12 08:46:04,698 - INFO - deleted item in tree at stack: (20, 31, 3, 1, 1) => #/texts/292\n",
      "2025-10-12 08:46:04,711 - INFO - deleted item in tree at stack: (20, 31, 3, 1, 1) => #/texts/292\n",
      "2025-10-12 08:46:04,724 - INFO - deleted item in tree at stack: (20, 31, 3, 1, 1) => #/texts/292\n",
      "2025-10-12 08:46:04,737 - INFO - deleted item in tree at stack: (20, 31, 3, 1, 1) => #/texts/292\n",
      "2025-10-12 08:46:04,750 - INFO - deleted item in tree at stack: (20, 31, 3, 1, 1) => #/texts/292\n",
      "2025-10-12 08:46:04,764 - INFO - deleted item in tree at stack: (20, 31, 3, 1, 1) => #/texts/292\n",
      "2025-10-12 08:46:04,778 - INFO - deleted item in tree at stack: (20, 31, 3, 1, 1) => #/texts/292\n",
      "2025-10-12 08:46:04,790 - INFO - deleted item in tree at stack: (20, 31, 3, 1, 1) => #/texts/292\n",
      "2025-10-12 08:46:04,803 - INFO - deleted item in tree at stack: (20, 31, 3, 1, 1) => #/texts/292\n",
      "2025-10-12 08:46:04,819 - INFO - deleted item in tree at stack: (20, 31, 3, 1, 1) => #/texts/292\n",
      "2025-10-12 08:46:04,834 - INFO - deleted item in tree at stack: (20, 31, 3, 1, 1) => #/texts/292\n",
      "2025-10-12 08:46:04,847 - INFO - deleted item in tree at stack: (20, 31, 3, 1, 1) => #/texts/292\n",
      "2025-10-12 08:46:04,863 - INFO - deleted item in tree at stack: (20, 31, 3, 1, 1) => #/texts/292\n",
      "2025-10-12 08:46:04,876 - INFO - deleted item in tree at stack: (20, 31, 3, 1, 1) => #/texts/292\n",
      "2025-10-12 08:46:04,891 - INFO - deleted item in tree at stack: (20, 31, 3, 1, 1) => #/texts/292\n",
      "2025-10-12 08:46:04,905 - INFO - deleted item in tree at stack: (20, 31, 3, 1, 1) => #/texts/292\n",
      "2025-10-12 08:46:04,919 - INFO - deleted item in tree at stack: (20, 31, 3, 1, 1) => #/texts/292\n",
      "2025-10-12 08:46:04,931 - INFO - deleted item in tree at stack: (20, 31, 3, 1, 1) => #/texts/292\n",
      "2025-10-12 08:46:04,944 - INFO - deleted item in tree at stack: (20, 31, 3, 1, 1) => #/texts/292\n",
      "2025-10-12 08:46:04,958 - INFO - deleted item in tree at stack: (20, 31, 3, 1, 1) => #/texts/292\n",
      "2025-10-12 08:46:04,970 - INFO - deleted item in tree at stack: (20, 31, 3, 1, 1) => #/texts/292\n",
      "2025-10-12 08:46:04,987 - INFO - deleted item in tree at stack: (20, 32, 0, 1, 1) => #/texts/310\n",
      "2025-10-12 08:46:05,003 - INFO - deleted item in tree at stack: (20, 32, 0, 1, 1) => #/texts/310\n",
      "2025-10-12 08:46:05,017 - INFO - deleted item in tree at stack: (20, 32, 0, 1, 1) => #/texts/310\n",
      "2025-10-12 08:46:05,031 - INFO - deleted item in tree at stack: (20, 32, 0, 1, 1) => #/texts/310\n",
      "2025-10-12 08:46:05,045 - INFO - deleted item in tree at stack: (20, 32, 0, 1, 1) => #/texts/310\n",
      "2025-10-12 08:46:05,059 - INFO - deleted item in tree at stack: (20, 32, 0, 1, 1) => #/texts/310\n",
      "2025-10-12 08:46:05,073 - INFO - deleted item in tree at stack: (20, 32, 0, 1, 1) => #/texts/310\n",
      "2025-10-12 08:46:05,087 - INFO - deleted item in tree at stack: (20, 32, 0, 1, 1) => #/texts/310\n",
      "2025-10-12 08:46:05,102 - INFO - deleted item in tree at stack: (20, 32, 0, 1, 1) => #/texts/310\n",
      "2025-10-12 08:46:05,119 - INFO - deleted item in tree at stack: (20, 32, 0, 1, 1) => #/texts/310\n",
      "2025-10-12 08:46:05,133 - INFO - deleted item in tree at stack: (20, 32, 0, 1, 1) => #/texts/310\n",
      "2025-10-12 08:46:05,148 - INFO - deleted item in tree at stack: (20, 32, 0, 1, 1) => #/texts/310\n",
      "2025-10-12 08:46:05,162 - INFO - deleted item in tree at stack: (20, 32, 0, 1, 1) => #/texts/310\n",
      "2025-10-12 08:46:05,177 - INFO - deleted item in tree at stack: (20, 32, 0, 1, 1) => #/texts/310\n",
      "2025-10-12 08:46:05,190 - INFO - deleted item in tree at stack: (20, 32, 0, 1, 1) => #/texts/310\n",
      "2025-10-12 08:46:05,205 - INFO - deleted item in tree at stack: (20, 32, 0, 1, 1) => #/texts/310\n",
      "2025-10-12 08:46:05,218 - INFO - deleted item in tree at stack: (20, 32, 0, 1, 1) => #/texts/310\n",
      "2025-10-12 08:46:05,232 - INFO - deleted item in tree at stack: (20, 32, 0, 1, 1) => #/texts/310\n",
      "2025-10-12 08:46:05,247 - INFO - deleted item in tree at stack: (20, 32, 0, 1, 1) => #/texts/310\n",
      "2025-10-12 08:46:05,262 - INFO - deleted item in tree at stack: (20, 32, 0, 1, 1) => #/texts/310\n",
      "2025-10-12 08:46:05,275 - INFO - deleted item in tree at stack: (20, 32, 0, 1, 1) => #/texts/310\n",
      "2025-10-12 08:46:05,290 - INFO - deleted item in tree at stack: (20, 32, 0, 1, 1) => #/texts/310\n",
      "2025-10-12 08:46:05,304 - INFO - deleted item in tree at stack: (20, 32, 0, 1, 1) => #/texts/310\n",
      "2025-10-12 08:46:05,318 - INFO - deleted item in tree at stack: (20, 32, 0, 1, 1) => #/texts/310\n",
      "2025-10-12 08:46:05,334 - INFO - deleted item in tree at stack: (20, 32, 0, 1, 1) => #/texts/310\n",
      "2025-10-12 08:46:05,350 - INFO - deleted item in tree at stack: (20, 32, 0, 1, 1) => #/texts/310\n",
      "2025-10-12 08:46:05,365 - INFO - deleted item in tree at stack: (20, 32, 0, 1, 1) => #/texts/310\n",
      "2025-10-12 08:46:05,380 - INFO - deleted item in tree at stack: (20, 32, 0, 1, 1) => #/texts/310\n",
      "2025-10-12 08:46:05,394 - INFO - deleted item in tree at stack: (20, 32, 0, 1, 1) => #/texts/310\n",
      "2025-10-12 08:46:05,410 - INFO - deleted item in tree at stack: (20, 32, 0, 1, 1) => #/texts/310\n",
      "2025-10-12 08:46:05,423 - INFO - deleted item in tree at stack: (20, 32, 0, 1, 1) => #/texts/310\n",
      "2025-10-12 08:46:05,442 - INFO - deleted item in tree at stack: (20, 32, 0, 1, 1) => #/texts/312\n",
      "2025-10-12 08:46:05,456 - INFO - deleted item in tree at stack: (20, 32, 0, 1, 1) => #/texts/312\n",
      "2025-10-12 08:46:05,470 - INFO - deleted item in tree at stack: (20, 32, 0, 1, 1) => #/texts/312\n",
      "2025-10-12 08:46:05,484 - INFO - deleted item in tree at stack: (20, 32, 0, 1, 1) => #/texts/312\n",
      "2025-10-12 08:46:05,499 - INFO - deleted item in tree at stack: (20, 32, 0, 1, 1) => #/texts/312\n",
      "2025-10-12 08:46:05,514 - INFO - deleted item in tree at stack: (20, 32, 0, 1, 1) => #/texts/312\n",
      "2025-10-12 08:46:05,526 - INFO - deleted item in tree at stack: (20, 32, 0, 1, 1) => #/texts/312\n",
      "2025-10-12 08:46:05,543 - INFO - deleted item in tree at stack: (20, 32, 0, 1, 1) => #/texts/312\n",
      "2025-10-12 08:46:05,543 - INFO - deleted item in tree at stack: (20, 32, 0, 1, 1) => #/texts/314\n",
      "2025-10-12 08:46:05,570 - INFO - deleted item in tree at stack: (20, 32, 0, 1, 1) => #/texts/314\n",
      "2025-10-12 08:46:05,579 - INFO - deleted item in tree at stack: (20, 32, 0, 1, 1) => #/texts/314\n",
      "2025-10-12 08:46:05,596 - INFO - deleted item in tree at stack: (20, 32, 0, 1, 1) => #/texts/314\n",
      "2025-10-12 08:46:05,610 - INFO - deleted item in tree at stack: (20, 32, 0, 1, 1) => #/texts/314\n",
      "2025-10-12 08:46:05,627 - INFO - deleted item in tree at stack: (20, 32, 0, 1, 1) => #/texts/314\n",
      "2025-10-12 08:46:05,643 - INFO - deleted item in tree at stack: (20, 32, 0, 1, 1) => #/texts/314\n",
      "2025-10-12 08:46:05,659 - INFO - deleted item in tree at stack: (20, 32, 0, 1, 1) => #/texts/314\n",
      "2025-10-12 08:46:05,676 - INFO - deleted item in tree at stack: (20, 32, 5, 1, 1) => #/texts/369\n",
      "2025-10-12 08:46:05,693 - INFO - deleted item in tree at stack: (20, 32, 5, 1, 1) => #/texts/369\n",
      "2025-10-12 08:46:05,710 - INFO - deleted item in tree at stack: (20, 32, 5, 1, 1) => #/texts/369\n",
      "2025-10-12 08:46:05,726 - INFO - deleted item in tree at stack: (20, 32, 5, 1, 1) => #/texts/369\n",
      "2025-10-12 08:46:05,743 - INFO - deleted item in tree at stack: (20, 32, 5, 1, 1) => #/texts/369\n",
      "2025-10-12 08:46:05,759 - INFO - deleted item in tree at stack: (20, 32, 5, 1, 1) => #/texts/369\n",
      "2025-10-12 08:46:05,776 - INFO - deleted item in tree at stack: (20, 32, 5, 1, 1) => #/texts/369\n",
      "2025-10-12 08:46:05,793 - INFO - deleted item in tree at stack: (20, 32, 5, 1, 1) => #/texts/369\n",
      "2025-10-12 08:46:05,812 - INFO - deleted item in tree at stack: (20, 32, 5, 1, 1) => #/texts/369\n",
      "2025-10-12 08:46:05,826 - INFO - deleted item in tree at stack: (20, 32, 5, 1, 1) => #/texts/369\n",
      "2025-10-12 08:46:05,843 - INFO - deleted item in tree at stack: (20, 32, 5, 1, 1) => #/texts/369\n",
      "2025-10-12 08:46:05,860 - INFO - deleted item in tree at stack: (20, 32, 5, 1, 1) => #/texts/369\n",
      "2025-10-12 08:46:05,878 - INFO - deleted item in tree at stack: (20, 32, 5, 1, 1) => #/texts/369\n",
      "2025-10-12 08:46:05,893 - INFO - deleted item in tree at stack: (20, 32, 5, 1, 1) => #/texts/369\n",
      "2025-10-12 08:46:05,910 - INFO - deleted item in tree at stack: (20, 32, 5, 1, 1) => #/texts/369\n",
      "2025-10-12 08:46:05,926 - INFO - deleted item in tree at stack: (20, 32, 5, 1, 1) => #/texts/369\n",
      "2025-10-12 08:46:05,946 - INFO - deleted item in tree at stack: (20, 32, 5, 1, 1) => #/texts/369\n",
      "2025-10-12 08:46:05,960 - INFO - deleted item in tree at stack: (20, 32, 5, 1, 1) => #/texts/369\n",
      "2025-10-12 08:46:05,976 - INFO - deleted item in tree at stack: (20, 32, 5, 1, 1) => #/texts/369\n",
      "2025-10-12 08:46:05,999 - INFO - deleted item in tree at stack: (20, 32, 5, 1, 1) => #/texts/369\n",
      "2025-10-12 08:46:06,011 - INFO - deleted item in tree at stack: (20, 32, 5, 1, 1) => #/texts/369\n",
      "2025-10-12 08:46:06,032 - INFO - deleted item in tree at stack: (20, 32, 5, 1, 1) => #/texts/369\n",
      "2025-10-12 08:46:06,043 - INFO - deleted item in tree at stack: (20, 32, 5, 1, 1) => #/texts/369\n",
      "2025-10-12 08:46:06,072 - INFO - deleted item in tree at stack: (20, 32, 5, 1, 1) => #/texts/369\n",
      "2025-10-12 08:46:06,077 - INFO - deleted item in tree at stack: (20, 32, 5, 1, 1) => #/texts/369\n",
      "2025-10-12 08:46:06,103 - INFO - deleted item in tree at stack: (20, 32, 5, 1, 1) => #/texts/369\n",
      "2025-10-12 08:46:06,114 - INFO - deleted item in tree at stack: (20, 32, 5, 1, 1) => #/texts/369\n",
      "2025-10-12 08:46:06,131 - INFO - deleted item in tree at stack: (20, 32, 5, 1, 1) => #/texts/369\n",
      "2025-10-12 08:46:06,143 - INFO - deleted item in tree at stack: (20, 32, 5, 1, 1) => #/texts/369\n",
      "2025-10-12 08:46:06,164 - INFO - deleted item in tree at stack: (20, 32, 5, 1, 1) => #/texts/369\n",
      "2025-10-12 08:46:06,186 - INFO - deleted item in tree at stack: (20, 32, 5, 1, 1) => #/texts/369\n",
      "2025-10-12 08:46:06,207 - INFO - deleted item in tree at stack: (20, 32, 5, 1, 1) => #/texts/369\n",
      "2025-10-12 08:46:06,219 - INFO - deleted item in tree at stack: (20, 32, 5, 1, 1) => #/texts/369\n",
      "2025-10-12 08:46:06,232 - INFO - deleted item in tree at stack: (20, 32, 5, 1, 1) => #/texts/369\n",
      "2025-10-12 08:46:06,249 - INFO - deleted item in tree at stack: (20, 32, 5, 1, 1) => #/texts/369\n",
      "2025-10-12 08:46:06,260 - INFO - deleted item in tree at stack: (20, 32, 5, 1, 1) => #/texts/369\n",
      "2025-10-12 08:46:06,276 - INFO - deleted item in tree at stack: (20, 32, 5, 1, 1) => #/texts/369\n",
      "2025-10-12 08:46:06,298 - INFO - deleted item in tree at stack: (20, 32, 5, 1, 1) => #/texts/369\n",
      "2025-10-12 08:46:06,316 - INFO - deleted item in tree at stack: (20, 32, 5, 1, 1) => #/texts/369\n",
      "2025-10-12 08:46:06,333 - INFO - deleted item in tree at stack: (20, 32, 5, 1, 1) => #/texts/369\n",
      "2025-10-12 08:46:06,349 - INFO - deleted item in tree at stack: (20, 32, 5, 1, 1) => #/texts/369\n",
      "2025-10-12 08:46:06,365 - INFO - deleted item in tree at stack: (20, 32, 5, 1, 1) => #/texts/369\n",
      "2025-10-12 08:46:06,381 - INFO - deleted item in tree at stack: (20, 32, 5, 1, 1) => #/texts/369\n",
      "2025-10-12 08:46:06,398 - INFO - deleted item in tree at stack: (20, 32, 5, 1, 1) => #/texts/369\n",
      "2025-10-12 08:46:06,415 - INFO - deleted item in tree at stack: (20, 32, 5, 1, 1) => #/texts/369\n",
      "2025-10-12 08:46:06,427 - INFO - deleted item in tree at stack: (20, 32, 5, 1, 1) => #/texts/369\n",
      "2025-10-12 08:46:06,456 - INFO - deleted item in tree at stack: (20, 32, 5, 1, 1) => #/texts/369\n",
      "2025-10-12 08:46:06,465 - INFO - deleted item in tree at stack: (20, 32, 5, 1, 1) => #/texts/369\n",
      "2025-10-12 08:46:06,483 - INFO - deleted item in tree at stack: (20, 32, 5, 1, 1) => #/texts/369\n",
      "2025-10-12 08:46:06,498 - INFO - deleted item in tree at stack: (20, 32, 5, 1, 1) => #/texts/369\n",
      "2025-10-12 08:46:06,511 - INFO - deleted item in tree at stack: (20, 32, 5, 1, 1) => #/texts/369\n",
      "2025-10-12 08:46:06,536 - INFO - deleted item in tree at stack: (20, 32, 5, 1, 1) => #/texts/369\n",
      "2025-10-12 08:46:06,548 - INFO - deleted item in tree at stack: (20, 32, 5, 1, 1) => #/texts/369\n",
      "2025-10-12 08:46:06,560 - INFO - deleted item in tree at stack: (20, 32, 5, 1, 1) => #/texts/369\n",
      "2025-10-12 08:46:06,585 - INFO - deleted item in tree at stack: (20, 32, 5, 1, 1) => #/texts/369\n",
      "2025-10-12 08:46:06,599 - INFO - deleted item in tree at stack: (20, 32, 5, 1, 1) => #/texts/369\n",
      "2025-10-12 08:46:06,616 - INFO - deleted item in tree at stack: (20, 32, 5, 1, 1) => #/texts/369\n",
      "2025-10-12 08:46:06,642 - INFO - deleted item in tree at stack: (20, 32, 5, 1, 1) => #/texts/369\n",
      "2025-10-12 08:46:06,650 - INFO - deleted item in tree at stack: (20, 32, 5, 1, 1) => #/texts/369\n",
      "2025-10-12 08:46:06,666 - INFO - deleted item in tree at stack: (20, 32, 5, 1, 1) => #/texts/369\n",
      "2025-10-12 08:46:06,682 - INFO - deleted item in tree at stack: (20, 32, 5, 1, 1) => #/texts/369\n",
      "2025-10-12 08:46:06,699 - INFO - deleted item in tree at stack: (20, 32, 5, 1, 1) => #/texts/369\n",
      "2025-10-12 08:46:06,715 - INFO - deleted item in tree at stack: (20, 32, 5, 1, 1) => #/texts/369\n",
      "2025-10-12 08:46:06,732 - INFO - deleted item in tree at stack: (20, 32, 5, 1, 1) => #/texts/369\n",
      "2025-10-12 08:46:06,760 - INFO - deleted item in tree at stack: (20, 32, 5, 1, 1) => #/texts/369\n",
      "2025-10-12 08:46:06,776 - INFO - deleted item in tree at stack: (20, 32, 5, 1, 1) => #/texts/369\n",
      "2025-10-12 08:46:06,792 - INFO - deleted item in tree at stack: (20, 32, 5, 1, 1) => #/texts/369\n",
      "2025-10-12 08:46:06,799 - INFO - deleted item in tree at stack: (20, 32, 5, 1, 1) => #/texts/369\n",
      "2025-10-12 08:46:06,815 - INFO - deleted item in tree at stack: (20, 32, 5, 1, 1) => #/texts/369\n",
      "2025-10-12 08:46:06,832 - INFO - deleted item in tree at stack: (20, 32, 5, 1, 1) => #/texts/369\n",
      "2025-10-12 08:46:06,859 - INFO - deleted item in tree at stack: (20, 32, 5, 1, 1) => #/texts/369\n",
      "2025-10-12 08:46:06,876 - INFO - deleted item in tree at stack: (20, 32, 5, 1, 1) => #/texts/369\n",
      "2025-10-12 08:46:06,876 - INFO - deleted item in tree at stack: (20, 32, 5, 1, 1) => #/texts/369\n",
      "2025-10-12 08:46:06,899 - INFO - deleted item in tree at stack: (20, 32, 5, 1, 1) => #/texts/369\n",
      "2025-10-12 08:46:06,915 - INFO - deleted item in tree at stack: (20, 32, 5, 1, 1) => #/texts/369\n",
      "2025-10-12 08:46:06,937 - INFO - deleted item in tree at stack: (20, 32, 5, 1, 1) => #/texts/369\n",
      "2025-10-12 08:46:06,943 - INFO - deleted item in tree at stack: (20, 32, 5, 1, 1) => #/texts/369\n",
      "2025-10-12 08:46:06,968 - INFO - deleted item in tree at stack: (20, 32, 5, 1, 1) => #/texts/369\n",
      "2025-10-12 08:46:06,983 - INFO - deleted item in tree at stack: (20, 32, 5, 1, 1) => #/texts/369\n",
      "2025-10-12 08:46:06,993 - INFO - deleted item in tree at stack: (20, 32, 5, 1, 1) => #/texts/369\n",
      "2025-10-12 08:46:07,015 - INFO - deleted item in tree at stack: (20, 32, 5, 1, 1) => #/texts/369\n",
      "2025-10-12 08:46:07,026 - INFO - deleted item in tree at stack: (20, 32, 5, 1, 1) => #/texts/369\n",
      "2025-10-12 08:46:07,043 - INFO - deleted item in tree at stack: (20, 32, 5, 1, 1) => #/texts/369\n",
      "2025-10-12 08:46:07,076 - INFO - deleted item in tree at stack: (20, 32, 5, 1, 1) => #/texts/369\n",
      "2025-10-12 08:46:07,096 - INFO - deleted item in tree at stack: (20, 32, 5, 1, 1) => #/texts/369\n",
      "2025-10-12 08:46:07,113 - INFO - deleted item in tree at stack: (20, 32, 5, 1, 1) => #/texts/369\n",
      "2025-10-12 08:46:07,128 - INFO - deleted item in tree at stack: (20, 32, 5, 1, 1) => #/texts/369\n",
      "2025-10-12 08:46:07,143 - INFO - deleted item in tree at stack: (20, 32, 5, 1, 1) => #/texts/369\n",
      "2025-10-12 08:46:07,160 - INFO - deleted item in tree at stack: (20, 32, 5, 1, 1) => #/texts/369\n",
      "2025-10-12 08:46:07,176 - INFO - deleted item in tree at stack: (20, 32, 5, 1, 1) => #/texts/369\n",
      "2025-10-12 08:46:07,193 - INFO - deleted item in tree at stack: (20, 32, 5, 1, 1) => #/texts/369\n",
      "2025-10-12 08:46:07,213 - INFO - deleted item in tree at stack: (20, 32, 5, 1, 1) => #/texts/369\n",
      "2025-10-12 08:46:07,226 - INFO - deleted item in tree at stack: (20, 32, 5, 1, 1) => #/texts/369\n",
      "2025-10-12 08:46:07,244 - INFO - deleted item in tree at stack: (20, 32, 5, 1, 1) => #/texts/369\n",
      "2025-10-12 08:46:07,259 - INFO - deleted item in tree at stack: (20, 32, 5, 1, 1) => #/texts/369\n",
      "2025-10-12 08:46:07,281 - INFO - deleted item in tree at stack: (20, 32, 5, 1, 1) => #/texts/369\n",
      "2025-10-12 08:46:07,293 - INFO - deleted item in tree at stack: (20, 32, 5, 1, 1) => #/texts/369\n",
      "2025-10-12 08:46:07,315 - INFO - deleted item in tree at stack: (20, 32, 5, 1, 1) => #/texts/369\n",
      "2025-10-12 08:46:07,327 - INFO - deleted item in tree at stack: (20, 32, 5, 1, 1) => #/texts/369\n",
      "2025-10-12 08:46:07,343 - INFO - deleted item in tree at stack: (20, 32, 5, 1, 1) => #/texts/369\n",
      "2025-10-12 08:46:07,360 - INFO - deleted item in tree at stack: (20, 32, 5, 1, 1) => #/texts/369\n",
      "2025-10-12 08:46:07,389 - INFO - deleted item in tree at stack: (20, 32, 5, 1, 1) => #/texts/369\n",
      "2025-10-12 08:46:07,425 - INFO - deleted item in tree at stack: (20, 32, 5, 1, 1) => #/texts/369\n",
      "2025-10-12 08:46:07,441 - INFO - deleted item in tree at stack: (20, 32, 5, 1, 1) => #/texts/369\n",
      "2025-10-12 08:46:07,460 - INFO - deleted item in tree at stack: (20, 32, 6, 1, 1) => #/texts/398\n",
      "2025-10-12 08:46:07,476 - INFO - deleted item in tree at stack: (20, 32, 6, 1, 1) => #/texts/398\n",
      "2025-10-12 08:46:07,503 - INFO - deleted item in tree at stack: (20, 32, 6, 1, 1) => #/texts/398\n",
      "2025-10-12 08:46:07,520 - INFO - deleted item in tree at stack: (20, 32, 6, 1, 1) => #/texts/398\n",
      "2025-10-12 08:46:07,539 - INFO - deleted item in tree at stack: (20, 32, 6, 1, 1) => #/texts/398\n",
      "2025-10-12 08:46:07,560 - INFO - deleted item in tree at stack: (20, 32, 6, 1, 1) => #/texts/398\n",
      "2025-10-12 08:46:07,577 - INFO - deleted item in tree at stack: (20, 32, 6, 1, 1) => #/texts/398\n",
      "2025-10-12 08:46:07,601 - INFO - deleted item in tree at stack: (20, 32, 6, 1, 1) => #/texts/398\n",
      "2025-10-12 08:46:07,631 - INFO - deleted item in tree at stack: (20, 32, 6, 1, 1) => #/texts/398\n",
      "2025-10-12 08:46:07,654 - INFO - deleted item in tree at stack: (20, 32, 6, 1, 1) => #/texts/398\n",
      "2025-10-12 08:46:07,677 - INFO - deleted item in tree at stack: (20, 32, 6, 1, 1) => #/texts/398\n",
      "2025-10-12 08:46:07,697 - INFO - deleted item in tree at stack: (20, 32, 6, 1, 1) => #/texts/398\n",
      "2025-10-12 08:46:07,718 - INFO - deleted item in tree at stack: (20, 32, 6, 1, 1) => #/texts/398\n",
      "2025-10-12 08:46:07,736 - INFO - deleted item in tree at stack: (20, 32, 6, 1, 1) => #/texts/398\n",
      "2025-10-12 08:46:07,757 - INFO - deleted item in tree at stack: (20, 32, 6, 1, 1) => #/texts/398\n",
      "2025-10-12 08:46:07,806 - INFO - Finished converting document PMC4136787 in 5.97 sec.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "# Mice in Bion-M 1 Space Mission: Training and Selection\n",
      "\n",
      "[Alexander Andreev-Andrievskiy](https://pubmed.ncbi.nlm.nih.gov/?term=%22Andreev-Andrievskiy%20A%22%5BAuthor%5D)\n",
      "\n",
      "1, 2, * , [Anfisa Popova](https://pubmed.ncbi.nlm.nih.gov/?term=%22Popova%20A%22%5BAuthor%5D)\n",
      "\n",
      "1, 2 , [Richard Boyle](https://pubmed.ncbi.nlm.nih.gov/?term=%22Boyle%20R%22%5BAuthor%5D)\n",
      "\n",
      "3 , [Jeffrey Alberts](https://pubmed.ncbi.nlm.nih.gov/?term=%22Alberts%20J%22%5BAuthor%5D)\n",
      "\n",
      "4 , [Boris Shenkman](https://pubmed.ncbi.nlm.nih.gov/?term=%22Shenkman%20B%22%5BAuthor%5D)\n",
      "\n",
      "1 , [Olga Vinogradova](https://pubmed.ncbi.nlm.nih.gov/?term=%22Vinogradova%20O%22%5BAuthor%5D)\n",
      "\n",
      "1 , [Oleg Dolgov](https://pubmed.ncbi.nlm.nih.gov/?term=%22Dolgov%20O%22%5BAuthor%5D)\n",
      "\n",
      "5 , [Konstantin Anokhin](https://pubmed.ncbi.nlm.nih.gov/?term=%22Anokhin%20K%22%5BAuthor%5D)\n",
      "\n",
      "5, 6 , [Darya Tsvirkun](https://pubmed.ncbi.nlm.nih.gov/?term=%22Tsvirkun%20D%22%5BAuthor%5D)\n",
      "\n",
      "1 , [Pavel Soldatov](https://pubmed.ncbi.nlm.nih.gov/?term=%22Soldatov%20P%22%5BAuthor%5D)\n",
      "\n",
      "1 , [Tatyana Nemirovskaya](https://pubmed.ncbi.nlm.nih.gov/?term=%22Nemirovskaya%20T%22%5BAuthor%5D)\n",
      "\n",
      "1 , [Eugeniy Ilyin](https://pubmed.ncbi.nlm.nih.gov/?term=%22Ilyin%20E%22%5BAuthor%5D)\n",
      "\n",
      "1 , [Vladimir Sychev](https://pubmed.ncbi.nlm.nih.gov/?term=%22Sychev%20V%22%5BAuthor%5D)\n",
      "\n",
      "1\n",
      "\n",
      "Editor: Nandu Goswami 7\n",
      "\n",
      "- Author information\n",
      "- Article notes\n",
      "- Copyright and License information\n",
      "\n",
      "1 Institute for Biomedical Problems, Russian Academy of Sciences, Moscow, Russia\n",
      "\n",
      "2 Moscow State University, Biology Faculty, Moscow, Russia\n",
      "\n",
      "3 Bio-Visualization, Imaging and Simulation Technology Center (BioVIS), NASA Ames Research Center, Moffett Field, California, United States of America\n",
      "\n",
      "4 Indiana University, Department of Psychological and Brain Sciences, Bloomington, Indiana, United States of America\n",
      "\n",
      "5 Anokhin Institute of Normal Physiology, Russian Academy of Medical Sciences, Moscow, Russia\n",
      "\n",
      "6 Kurchatov NBIC-center, National Research Centre \"Kurchatov Institute\", Moscow, Russia\n",
      "\n",
      "7 Medical University of Graz, Austria\n",
      "\n",
      "\n",
      "\n",
      "* E-mail: aandrievsky@gmail.com\n",
      "\n",
      "**Competing Interests:** The authors have declared that no competing interests exist.\n",
      "\n",
      "Conceived and designed the experiments: AAA BS EI OV OD KA PS TN VS RB JA. Performed the experiments: AAA AP OV OD DT PS TN VS JA. Analyzed the data: AAA AP EI OD PS VS JA. Contributed reagents/materials/analysis tools: AAA BS EI OV OD KA PS VS RB JA. Wrote the paper: AAA AP BS EI OV RB VS JA.\n",
      "\n",
      "#### Roles\n",
      "\n",
      "**Nandu Goswami** : Editor\n",
      "\n",
      "Received 2013 Dec 25; Accepted 2014 Jul 17; Collection date 2014.\n",
      "\n",
      "This is an open-access article distributed under the terms of the Creative Commons Public Domain declaration, which stipulates that, once placed in the public domain, this work may be freely reproduced, distributed, transmitted, modified, built upon, or otherwise used by anyone for any lawful purpose.\n",
      "\n",
      "[PMC Copyright notice](\\about\\copyright)\n",
      "\n",
      "PMCID: PMC4136787  PMID: [25133741](https://pubmed.ncbi.nlm.nih.gov/25133741/)\n",
      "\n",
      "## Abstract\n",
      "\n",
      "After a 16-year hiatus, Russia has resumed its program of biomedical research in space, with the successful 30-day flight of the Bion-M 1 biosatellite (April 19-May 19, 2013). The principal species for biomedical research in this project was the mouse. This paper presents an overview of the scientific goals, the experimental design and the mouse training/selection program. The aim of mice experiments in the Bion-M 1 project was to elucidate cellular and molecular mechanisms, underlying the adaptation of key physiological systems to long-term exposure in microgravity. The studies with mice combined *in vivo* measurements, both in flight and post-flight (including continuous blood pressure measurement), with extensive *in vitro* studies carried out shortly after return of the mice and in the end of recovery study. Male C57/BL6 mice group housed in space habitats were flown aboard the Bion-M 1 biosatellite, or remained on ground in the control experiment that replicated environmental and housing conditions in the spacecraft. Vivarium control groups were used to account for housing effects and possible seasonal differences. Mice training included the co-adaptation in housing groups and mice adaptation to paste food diet. The measures taken to co-adapt aggressive male mice in housing groups and the peculiarities of \"space\" paste food are described. The training program for mice designated for *in vivo* studies was broader and included behavioral/functional test battery and continuous behavioral measurements in the home-cage. The results of the preliminary tests were used for the selection of homogenous groups. After the flight, mice were in good condition for biomedical studies and displayed signs of pronounced disadaptation to Earth's gravity. The outcomes of the training program for the mice welfare are discussed. We conclude that our training program was effective and that male mice can be successfully employed in space biomedical research.\n",
      "\n",
      "## Introduction\n",
      "\n",
      "After a 16-year hiatus, Russia resumed in 2013 its program of biomedical research in space, with the successful 30-day flight of the Bion-M 1 biosatellite (April 19-May 19, 2013), a specially designed automated spacecraft dedicated to life-science experiments. \"M\" in the mission's name stands for \"modernized\"; the epithet was equally applicable to the spacecraft and the research program. The principal animal species for physiological studies in this mission was the mouse ( *Mus musculus* ). Unlike more recent space experiments that used female mice, males were flown in the Bion-M 1 mission. The challenging task of supporting mice in space for this unmanned, automated, 30-day-long mission, was made even more so by the requirement to house the males in groups.\n",
      "\n",
      "Russian biomedical research in space traditionally has employed dogs, rats, monkeys, and more recently Mongolian gerbils. The flight of Laika in 1957 was one of the early dog experiments and became world famous for demonstrating that a living organism can withstand rocket launch and weightlessness, thus paving the way for the first human spaceflight. Laika's success also promoted biomedical research with other non-human animals in space that culminated with the Bion biosatellites program. A total of 212 rats and 12 monkeys were launched on 11 satellites and exposed in microgravity for 5.0-22.5 days between 1973 and 1997. Animal experiments on the Bion missions have contributed comprehensive data on adaptive responses of sensorimotor, cardiovascular, muscle, bone and other systems to spaceflight conditions and the mechanisms underlying these adaptations [[1]](#pone.0104830-Grigoriev1) , [[2]](#pone.0104830-Ilyin1) .\n",
      "\n",
      "The use of mice for space experiments offers numerous advantages. Probably the most apparent one is their small size and thus the possibility of utilizing more animals per flight, thus increasing scientific output and the cost-efficiency ratio. Comparisons of data obtained with mice, with those obtained from larger species or humans can also reveal how factors affecting adaptation to spaceflight conditions depend on the size of the organism. The mouse has become the most prevalent \"mammalian model\" in biomedical research, with a fully described genome and an established role in genetically engineered mutants. While mice are preferred mammalian models for molecular biology studies, their small size is a debated limitation rather than an advantage for physiological studies. Miniaturization of scientific hardware has reduced some of the disadvantages of the species small size. Finally, the use of genetically controlled mice offers a means to reduce inter-individual variability and obtain potentially more consistent results.\n",
      "\n",
      "Despite the advantages of the mouse as a model organism for space research, their use was rather limited (apart from a number of experiments with mice during early space exploration in the 1950's and 1960's, which were aimed primarily at testing if living organisms can survive the launch or a brief exposure in microgravity) [[3]](#pone.0104830-Burgess1) . Flight experiments with mice were performed aboard STS-90 (\"NeuroLab\"), STS-108, STS-129, STS-131 (\"Mouse Immunology I\"), STS-133 (\"Mouse Immunology II\"), and STS-135 with exposure times ranging from 12 to 16 days. Research programs of these flights were largely focused on studies of muscle, bone/tendon/cartilage, nervous, and cardiovascular systems, and innate and acquired immune responses. Experiments were performed with groups of 30 or fewer female C57BL/6J mice, which were dissected typically shortly after return [[4]](#pone.0104830-Buckey1) - [[9]](#pone.0104830-Lujan1) .\n",
      "\n",
      "The Mice Drawer System (MDS) experiment of the Italian Space Agency is by far the longest spaceflight of mice to date [[10]](#pone.0104830-Cancedda1) . In this mission, 6 mice were exposed for 91 days aboard the International Space Station. The advantages offered by the possibility of genetic manipulations with mice were utilized in this experiment; three mice were transgenic with pleiotrophin overexpression (C57BLJ10/ PTN) and three mice were their wild-type counterparts. The MDS habitats required periodic replenishment and servicing by by astronauts. Sadly, half of the mice died during the course of this mission due to various estimated reasons.\n",
      "\n",
      "In the present paper we aim to present a brief overview of the Bion-M 1 mission scientific goals and experimental design. Of particular interest we will focus on the program of mouse training and selection for the experiments, and some outcomes of the Bion-M 1 mission.\n",
      "\n",
      "### Overview of the Bion-m 1 mission\n",
      "\n",
      "The scientific program of the Bion-M 1 project was aimed at obtaining data on mechanisms of adaptation of muscle, bone, cardiovascular, sensorimotor and nervous systems to prolonged exposure in microgravity and during post-flight recovery. To this aim, functional measurements *in vivo* were combined with complementary morphological, biochemical, cellular and molecular studies performed *in vitro* . To our knowledge, Bion-M 1 is the first study with mice to follow the dynamics of re-adaptation to normal gravity or to assess post-flight *in vivo* functions.\n",
      "\n",
      "The *in vivo* measurements in-flight included video recording of mice behavior and continuous recording of arterial pressure during all stages of the flight. After the flight, testing of animal behavior, memory and learning, vestibular function, physical activity, strength and fitness, as well as continuous registration of cardiovascular parameters were performed. The set of *in vivo* measurements was designed to provide data on the effects of microgravity on the physiological systems of interest, and to follow the time course of re-adaptation to Earth's gravity after the flight. To reduce the possibility of inadvertently stressing the mice with numerous post-flight tests and thus obscuring the effects of spaceflight, the decision was made to monitor mice in their home cage using an automated data collection system (TSE PhenoMaster) and implantable telemetry, while reducing the number of tests [[11]](#pone.0104830-AndreevAndrievskiy1) . The program had to be performed with a limited number of animals; thus the use of a repeated measures design. Mice were tested before and after the flight (or its simulation on-ground) [[12]](#pone.0104830-Festing1) .\n",
      "\n",
      "The *in vitro* studies were focused on metabolism and the structure and function of tissues from microgravity-exposed mice. More specifically, the individual studies were selected to maximize the scientific yield and targeted the impact of microgravity and other factors specific to spaceflight on biochemical, morphological and, primarily, cellular and molecular mechanisms involved in the adaptation of main physiological systems, such as the central nervous system, sensory systems, skeletal muscle, supporting tissues, including bone and connective tissues, cardiovascular, respiratory, digestive, neuroendocrine and immune systems.\n",
      "\n",
      "The following organs and tissues were harvested: brain structures, vestibular organs, retina and crystalline lens, spinal cord, muscles of hind limbs and back, vertebral column, fore and hind paw bones, articular and cartilage structures, ligaments, bone marrow, cardiac muscle, arteries from brain and hind legs, lung, intestines, liver, kidney, spleen, thymus, skin, salivary glands, testis and blood. These samples were analyzed using morphological (light and electron microscopy, histochemistry and immunohistochemistry), biochemical and a variety of molecular-biology techniques (enzyme-linked immune assays, high-pressure liquid chromatography, western-blot, real-time PCR, whole-genome gene expression). Functional properties of skeletal muscle, cardiac muscle and resistive arteries samples were evaluated using myography and atomic force microscopy. Mice for *in vitro* studies were euthanized by cervical dislocation; because of the diversity and number of studies, this method was chosen as the common procedure to reduce variability and one that provided the least interference with the objectives of the *in vitro* studies.\n",
      "\n",
      "## Materials and Methods\n",
      "\n",
      "### Ethical statement\n",
      "\n",
      "The study was approved by IACUC of MSU Institute of Mitoengineering (Protocol  35, 1 November, 2012) and of Biomedical Ethics Commission of IBMP (protocol  319, 4 April, 2013) and conducted in compliance with the European Convention for the Protection of Vertebrate Animals used for Experimental and Other Scientific Purposes [[13]](#pone.0104830-European1) .\n",
      "\n",
      "### Animals\n",
      "\n",
      "Experiments were performed with C57BL/6N male mice. The choice of strain was based on the requirements of the tissue harvesting program participants and the fact that C57/BL6 is one of the most widely used strains. The selection of male mice was based on the same requirement. The advantages offered by using males are their substantially larger size and the absence of sex steroid cycling compared to females; the vast majority of animal research is conducted using males, so their use in this study made the results more readily comparable to the data in the published literature. Nevertheless, the risks associated with group-housing mature males were recognized and greatly affected the preparatory procedures. It was decided that the optimal age of mice at launch was 4-5 months, when the males were still young but fully mature. The growth rate in mice is known to slow down by this age, and obesity, which C57/BL6 mice are prone to exhibit, is not yet considered a factor.\n",
      "\n",
      "Space environment affects microorganisms by increasing their virulence; major shifts in immune status in space are well documented [[5]](#pone.0104830-Pecaut1) , [[8]](#pone.0104830-Baqai1) . It is reasonable to think that even pseudo pathogenic species may increase morbidity during prolonged exposure in space [[14]](#pone.0104830-Aviles1) , [[15]](#pone.0104830-Wilson1) . These considerations underscored the necessity for the use of specific pathogen-free mice in the experiments, a side benefit of this approach being the reduced variability of data obtained in SPF-animals [[16]](#pone.0104830-Howard1) .\n",
      "\n",
      "Male C57BL/6N mice (n=300) weighing 22-25 g were purchased from the Animal Breeding Facility - Branch of Shemyakin &amp; Ovchinnikov Institute of Bioorganic Chemistry. Mice were specific pathogen free. The main site for mouse training was the animal facility of Moscow State University's Institute of Mitoengineering. At the time of launch and the start of the related control experiments, the mice were 19-20 weeks old.\n",
      "\n",
      "Mice were individually identified using RFID chips (El1000, Felixcan, Spain) and earmarks. Routine manipulations of the mice consisted of daily handling, examination, and weighing.\n",
      "\n",
      "### Experimental groups\n",
      "\n",
      "The Bion-M 1 biosatellite, launched on April 19, 2013 from cosmodrome Baikonur, and the descent module landed on May 19, 2013 in the vicinity of Orenburg, successfully fulfilling the plan for an unmanned, 30-day-long orbital spaceflight. Housing and climate parameters were replicated in the subsequent ground control (GC) experiment (July 26 to August 26, 2013). A total of 4 experimental groups were used for the flight and ground control experiments (n=45 per group). An additional fifth group included the backup mice for the main flight group (n=45). Mice of the space flight group (SF) were exposed to microgravity for 30 days. Concurrent with the SF mice, another group of 45 mice remained in the animal facility (SFV). The ground control (GC) experiment was conducted, after the landing, in the refurbished BOS flight habitats. The habitats were installed in a climatic chamber that replicated the temperature, humidity, gas composition and other flight-specific climate parameters. The corresponding vivarium control (GCV) mice were housed in the animal facility. The separate and concurrent GCV groups were used to account for possible seasonal differences between SF and GC mice.\n",
      "\n",
      "Each of the groups (SF, GC, SFV, GCV and backup SF mice) included mice designated for *in vivo* studies and recovery (n=10) and mice for dissection and *in vitro* measurements (n=35). Each *in vivo* study subgroup, in its turn, consisted of 5 mice implanted with telemetry probes to monitor blood pressure and 5 intact animals.\n",
      "\n",
      "Mice were handled and trained before the flight and ground control experiments. Basically, training consisted of shaping the groups of three mice each for social housing and adaptation to paste diet. The training of mice designated for *in vivo* studies was more comprehensive. It started with implanting the telemetry probes and, following recovery, a set of preliminary behavioral and functional tests ( [Figure 1](#pone-0104830-g001) , [Table 1](#pone-0104830-t001) and [2](#pone-0104830-t002) ).\n",
      "\n",
      "Mice of SF, SFV and backup SF groups were co-adapted in housing groups of 3 mice each, SF mice were adapted to paste food diet, and mice of in vivo subgroups passed through preliminary tests. After transportation to and adaptation at Baikonur, SF mice were flown aboard the Bion-M 1 satellite for 30 days. After landing mice were examined and transported to Moscow, where animals of the in vitro subgroup were dissected, while recovery dynamics was followed in the in-vivo subgroup before dissection 7 days after landing. Ground control experiment replicated the principal stages of the spaceflight experiment.\n",
      "\n",
      "<!-- image -->\n",
      "\n",
      "#### Table 1. Key dates of the flight experiment.\n",
      "\n",
      "| FLIGHT EXPERIMENT: key dates                              | Time relative to Launch (L) or Return (R)                 |\n",
      "|-----------------------------------------------------------|-----------------------------------------------------------|\n",
      "| Arrival of mice from the breeding facility                | L-51d                                                     |\n",
      "| RFID-chips implantation, ear marking                      | L-50d                                                     |\n",
      "| Telemetry probes implantation                             | L-50d                                                     |\n",
      "| Weighing, examination                                     | L-50d - L-3d                                              |\n",
      "| Co-adaptation of mice in groups of three                  | L-45d - L-3d                                              |\n",
      "| Adaptation of mice to paste diet                          | L-14d - L-3d                                              |\n",
      "| Training/tests with mice designated for in vivo studies   | L-30d - L -9d                                             |\n",
      "| Transportation to Baikonur                                | L-7d                                                      |\n",
      "| Adaptation at launch site                                 | L-7d - L-3d                                               |\n",
      "| Final selection of the flight group                       | L -3d                                                     |\n",
      "| Transfer of mice to the habitats                          | L-3d                                                      |\n",
      "| Installation of habitat assemblies into the biosatellite  | L-3d                                                      |\n",
      "| Monitoring of mice in habitats                            | L-3d - L-1d                                               |\n",
      "| Transportation of the biosatellite for further assembling | L-2d                                                      |\n",
      "| Transportation to launch site                             | L-1d                                                      |\n",
      "| Installation of the rocket at launch table                | L-16h                                                     |\n",
      "| Launch  2013/04/19 14:00                                 | L 0                                                       |\n",
      "| Bion-M 1 flight  477 orbits  30 days (2013/04/19-05/19) | Bion-M 1 flight  477 orbits  30 days (2013/04/19-05/19) |\n",
      "| Braking engine start                                      | R -0.75h                                                  |\n",
      "| Atmospheric entry                                         | R-0.25h                                                   |\n",
      "| Parachute opening                                         | R-0.17h                                                   |\n",
      "| Touchdown engines firing (2013/05/19 07:11)               | R 0                                                       |\n",
      "| Recovery team arrival                                     | R +0.25h                                                  |\n",
      "| Dismounting of habitat assemblies                         | R+2h-R+3h                                                 |\n",
      "| Mice examination at recovery site                         | R+2h  R+5h                                               |\n",
      "| Transportation to Moscow                                  | R+5h-R+12h                                                |\n",
      "| Euthanasia and in vitro studies                           | R+13h  R+25h                                             |\n",
      "| In vivo tests and recovery monitoring                     | R+0d  R+7d                                               |\n",
      "| Euthanasia of the recovery group, in vitro studies        | R+7d                                                      |\n",
      "\n",
      "[Open in a new tab](table\\pone-0104830-t001)\n",
      "\n",
      "#### Table 2. Key dates of the on-ground control experiment.\n",
      "\n",
      "| CONTROL EXPERIMENT: key dates                               | Time relative to experiment Start (S) or End (E)       |\n",
      "|-------------------------------------------------------------|--------------------------------------------------------|\n",
      "| Arrival of mice from the breeding facility                  | S-38d                                                  |\n",
      "| RFID-chips implantation, ear marking                        | S-37d                                                  |\n",
      "| Telemetry probes implantation                               | S-36d                                                  |\n",
      "| Weighing, examination                                       | S-37d  S-2d                                           |\n",
      "| Co-adaptation of mice in groups of three                    | S-29d  S-2d                                           |\n",
      "| Adaptation of mice to paste diet                            | S-14d  S-2d                                           |\n",
      "| Training/tests with mice designated for in vivo studies     | S-21d  S-6d                                           |\n",
      "| Transportation to IBMP                                      | S-1d                                                   |\n",
      "| Transfer of mice to the habitats                            | S-1d                                                   |\n",
      "| Installation of habitat assemblies into the climate chamber | S-1d                                                   |\n",
      "| Monitoring of mice in habitats                              | S-1d-S0                                                |\n",
      "| Control experiment start (2013/07/26 14:00)                 | S0                                                     |\n",
      "| Ground control experiment  30 days (2013/07/26-08/26)      | Ground control experiment  30 days (2013/07/26-08/26) |\n",
      "| Control experiment end (2013/08/26 09:00)                   | E0                                                     |\n",
      "| Dismounting of habitat assemblies                           | E+1h  E+2h                                            |\n",
      "| Mice examination                                            | E+1h  E+3h                                            |\n",
      "| Transportation imitation                                    | E+4h  E+8h                                            |\n",
      "| Euthanasia and in vitro studies                             | E+8h  E+23h                                           |\n",
      "| In vivo tests and recovery monitoring                       | E+0d  E+7d                                            |\n",
      "| Euthanasia of the recovery group, in vitro studies          | E+7d                                                   |\n",
      "\n",
      "[Open in a new tab](table\\pone-0104830-t002)\n",
      "\n",
      "### Housing\n",
      "\n",
      "The BOS habitats were designed to house up to three mice per group [[17]](#pone.0104830-VanLoo1) . Stable cohorts of three mice were maintained throughout the training program and consequent experiments, with the exception of brief periods of post-surgery recovery (5-6 days) and behavioral tests (7 days), when they were housed individually.\n",
      "\n",
      "Individually ventilated cages (GM 500, Techniplast, floor area 500 cm 2 ) were used. Techniplast mice shelters (red plastic triangles, 15Lx11Wx8H cm), paper tubes and paper tissues for nesting material were provided to enrich the environment. Cage cleaning was done once weekly, and water was refreshed twice weekly. All materials were sterilized prior to use by autoclave. Techniques aimed to prevent animal contamination were thoroughly followed.\n",
      "\n",
      "### Habitats\n",
      "\n",
      "The hardware for mice housing in-flight was an adapted version of the habitats used for rats in the original Bion program. It was developed by Biofizpribor, according to the IBMP requirements and was tested by IBMP in 2011. Each block consisted of five individual habitats (BOS, Block Obespecheniya Soderzhaniya, Russian for - literally - Unit for the Provision of Housing). The living compartment of each habitat was an acrylic cylinder 982 mm in diameter and 20010 mm in length with a total volume of approximately 1.7 l ( [Figure 2](#pone-0104830-g002) ). A stainless steel feeder (9 ml) was mounted at one end (the \"top\") of the cylinder. The feeder had a flap, used in the refill procedure. The food distribution system (except for the feeders) was common to all five habitats in the block. Paste \"space\" food was pumped from the container and distributed between individual habitats of the assembly 6 times daily (every 4 hours at 00:00, 04:00, 08:00, 12:00, 16:00 and 20:00), a total of 54 g/day per 3 mice. Food distribution was adjusted through manipulations of pressure developed in the container and in the feeders to follow this pattern prior to experiments.\n",
      "\n",
      "Figure 2\n",
      "\n",
      "<!-- image -->\n",
      "\n",
      "Bulbs for day-phase (08:00-20:00, 455 Lux) and night-phase (20:00-08:00, 52 Lux) lighting, an air supply fan (0.3 m/s) and an air distribution system were positioned on one side of the cylinder. A video camera was installed between the light bulbs; video was recorded 6 times daily (simultaneously with feeding, each session lasted 2 hours) at 8 frames per second. The opposite side of the cylinder had the grid for waste removal. The waste removal system was individual for each habitat. The door at the \"bottom\" of the cylinder had a glass window. In order to minimize impact of variability arising from possible differences in mice housing conditions, experimental subgroups were formed from mice that were housed in different habitats.\n",
      "\n",
      "Air composition and climate parameters in the satellite were maintained by the centralized life support system and remained in the designed range throughout the flight.\n",
      "\n",
      "### Diet\n",
      "\n",
      "During the initial stage of training, mice were provided with standard pelleted food (Assortiment Agro, Russia) and deionized water given *ad libitum* . Two weeks prior to launch, and the start of the GC experiment, the animals were adapted to the flight paste food diet. Both the paste and standard diets (pelleted food and water) were first presented together; after 2-3 days the water bottle was removed, followed in another 2-3 days by the removal of the pelleted food. Vivarium control (SFV and GCV) groups were fed standard chow and water throughout the experiments. The \"space\" paste food diet was developed at IBMP by T. Gurieva and E. Mednikova. It was produced based on the standard chow, with water and casein as gelling agent added.\n",
      "\n",
      "Consumption of standard pelleted chow and water (in SFV and GCV) or paste diet (in SF and GC) was monitored daily for ten days in ten randomly chosen cages per group. Data for the cage was averaged across the days and expressed as g per 10 g bodyweight.\n",
      "\n",
      "A separate preliminary experiment in metabolic chambers was conducted to quantify urination in mice fed standard chow and paste food. Four groups of mice, of 3 mice each, were housed in metabolic chambers (Techniplast, Italy) and fed pelleted food (grinded) or paste food. Body weight, food and water consumption, urine volume and feces weight were monitored daily (3 days for each diet).\n",
      "\n",
      "### Telemetry probes implantation\n",
      "\n",
      "Arterial pressure was monitored in sub-groups of mice designated for recovery, using implantable probes and telemetry hardware (Data Science International, USA). The hardware was adapted and integrated to the Bion-M biosatellite by engineers from Biofizpribor (St. Petersburg, Russia) and the Centre national d'tudes spatiales (CNES, France). Miniature probes (model PA-C10) weighing 1.4 g were implanted 3 weeks before the start of experiments, following the manufacturer's instructions. The catheter of the probe was implanted into the left carotid so that its tip just reached the aorta [[18]](#pone.0104830-Kurtz1) . The body of the transmitter was implanted subcutaneously on the flank, briefly described below.\n",
      "\n",
      "Mice were anaesthetized by an intraperitoneal injection of a solution containing tiletamine/zolazepam (15 mg/kg each) combined with xylazine (3 mg/kg). Depth of anesthesia was assessed regularly by hind leg pinch and observation of respiration rate, and maintained by additional dose(s) of the mixture (about 20% of the initial dose) as needed. A midline incision was made on the ventral neck. The carotid artery was separated and ligated 1-2 mm cranially to its bifurcation. A second ligature was used to suspend the artery 4-5 mm proximal to the site of ligation. Through a small incision in the artery wall, the catheter was advanced 6-7 mm towards the aorta and fixed in place with sutures and acrylic glue. A subcutaneous pocket on the flank of the animal was made with blunt scissors, and the transmitter was inserted there with a small hemostat. The catheter and the transmitter were fixed in place with acrylic glue and the skin incision closed with absorbable sutures. During the recovery period, mice were housed individually; they were under daily veterinarian control and received ibuprofen 4 mg/ml and Bactrim 4 mg/ml in drinking water.\n",
      "\n",
      "### Pre-flight training of mice designated for in vivo studies\n",
      "\n",
      "Mice for the *in vivo* studies were trained and tested before the experiments to select homogenous groups and to benefit from repeated measures in experimental design; the latter consideration was particularly important in light of relatively small number of animals.\n",
      "\n",
      "On the morning of day 0 mice a battery of behavioral and functional tests was administered. First, animals were examined for visible abnormalities in exterior and simple reflexes [[19]](#pone.0104830-Schneider1) . *The Open field test* was performed in a round, 60 cm arena of black plastic with a grey floor and 40 cm high walls [[20]](#pone.0104830-Gould1) . Mouse behavior was video recorded at 25 fps for 10 minutes under a bright white light (200 Lux) followed by 10 min in darkness (IR light). Videos were analyzed for locomotor activity and other behavioral parameters using EthoVision v. 8.5 (Noldus). *Grip force* was measured using Grass FT-03 force transducer [[21]](#pone.0104830-Maurissen1) . Analog output was amplified, digitized at 1 kHz sampling rate and analyzed using PowerGraph software (InteropticaS, Russia). *RotaRod* test was conducted using UgoBasil hardware with rotation speed increased from 6 to 50 rpm in 3 min, 5 trials were performed with each mouse [[22]](#pone.0104830-Rogers1) .\n",
      "\n",
      "On the evening of the same day, animals were placed into cages of *Phenomaster* (TSE Instruments, Germany), an automated data collection system, where they were monitored for 7 days. The system was equipped with an infrared grid to register animal activity, scales for the water bottle and feeder, a running wheel with unrestricted access and rotation counter, an \"operant wall\" for positively reinforced operant conditioning [[23]](#pone.0104830-Malkki1) . There were 16 such cages, housing mice individually. Measures were read once per minute using TSE software. Blood pressure and heart rate in mice, instrumented with telemetry probes were continuously recorded throughout the tests.\n",
      "\n",
      "*Operant conditioning* sessions were conducted automatically, once a day and started at 22:30, controlled by IntelliMaze software control. The first session was designed to reinforce nosepokes emitted by an animal while both light stimuli (left and right) were active. This session was used to establish individual baseline tendencies to display nosepokes to the right or left. On subsequent days, mice were trained to direct nosepokes to the side opposite to their initial bias. The number of rewarded responses was limited to 50; each sweetened dry milk pellet weighed 14 mg. To promote learning, mice were maintained at 90% ad lib body weight. Access to food was provided throughout the day from 07:30 until 19:00-20:00. Reward pellets were the alternative source of food.\n",
      "\n",
      "The quality of blood pressure signal (for the operated mice), probabilities of running wheel use, combined with performance in conditioning task were used as criteria for inclusion into experimental groups. Selected mice were randomly assigned to either flight or ground control or vivarium groups.\n",
      "\n",
      "### Transportation\n",
      "\n",
      "The groups of mice designated for flight were transported to Cosmodrome Baikonur one week before launch. Transportation was by car and by air in several steps and lasted approximately 24 h. Containers of nontransparent blue plastic 4060 cm with 4 sub-compartments (each for a group of three mice) equipped with air filters (Animal Breeding Facility - Branch of Shemyakin &amp; Ovchinnikov Institute of Bioorganic Chemistry) were used. Paste food was provided to mice during transportation.\n",
      "\n",
      "### Statistical analysis\n",
      "\n",
      "One-way ANOVA (transportation bodyweight data) followed by Bonferroni's multiple comparisons was performed with transportation bodyweight data. In order the pre/post flight bodyweight data set met the assumptions of normality and homoscedasticity intrinsic to the ANOVA model, rank transformation was applied to the entire set of observations [[24]](#pone.0104830-Conover1) prior to two-way ANOVA followed by Bonferroni post-hoc comparisons; noteworthy, the results of the test on raw and transformed data were identical. Fisher's exact test (survival), Man-Whitney U-test (home cage food and water consumption, open field test behavioral data) or paired Student's t-test (metabolic chambers data) was performed using GraphPad Prism software (version 6.00 for MacOS, GraphPad Software, USA).\n",
      "\n",
      "## Results and Discussion\n",
      "\n",
      "### Group housing\n",
      "\n",
      "Living conditions for animals considered optimal on earth cannot be provided in the confines of a space satellite. Microgravity, apart from purely technical limitations of the housing hardware, may disrupt normal mouse behavior, and an inability to express natural behavior is posited as a potential cause of distress in laboratory animals [[25]](#pone.0104830-Newberry1) . This and more obvious negative factors, such as accumulated waste, suboptimal feeding regimen etc., can negatively affect mice and could increase aggressiveness in space habitats. As a result we considered aggressive interactions a major risk factor. To minimize this risk, we specifically sought to shape the groups for social housing and carefully select stable groups prior to experiments.\n",
      "\n",
      "From an initial set of 300 male mice, 88 groups of 3 mice were formed, a total of 53 groups were used for the space flight (SF) experiment and corresponding SFV, 35 groups for subsequent GC and GCV studies (87%). During the co-adaptation period, a total of 12 groups had to be rearranged (14%).\n",
      "\n",
      "Male mice are known to have aggressive tendencies. Their aggressiveness depends on a number of factors, including strain (C57/BL6 is considered one of the more aggressive strains), living space and other housing conditions and, in the case of group housing, the history of group interactions is a major factor [[17]](#pone.0104830-VanLoo1) . In an animal facility, aggressive interactions are easily overcome by simply removing the aggressor from a group or by individual housing of mature males. Individual housing, however, precludes all social interactions and thus compromises the animals' overall welfare. For this reason, environmental enrichment is often employed in animal facilities, with the goal of reducing aggressive interactions.\n",
      "\n",
      "While the analysis of measures promoting co-habitation of male mice was beyond the scope of the current project, several pertinent observations were made. We reasoned that environmental enrichment and daily handling were critically important for successful co-adaptation of mice and the development of sustainable hierarchy in the group. In a preliminary experiment, when no environmental enrichment was used, only 50% of mice (out of a sample of 42) successfully co-adapted to living in groups of three. The enrichment used to prepare mice for the Bion-M 1 experiments (shelters, nesting material and paper tubes) proved to be useful in two ways. Shelter with good qualities is salient to mice, and male mice will compete for the resource [[26]](#pone.0104830-VandeWeerd1) . Additionally, adequate shelter facilitates the observation of nesting behavior within the group. Nesting behavior can differentiate groups, with stable groups building a communal nest within a chamber, whereas mice that cannot co-adapt as a group compete and fight for the shelter, with one or two mice building a separate nest outside of the chamber.\n",
      "\n",
      "Handling and other simple manipulations are known to induce an acute stress reaction in mice, seen as an increase in blood pressure, heart rate, body temperature, and plasma corticosterone [[27]](#pone.0104830-Bouwknecht1) -. At the same time, handling is often used to habituate mice and rats to experimental manipulations (mild stress), with different handling techniques having different impacts on their behavioral indices [[31]](#pone.0104830-Hurst1) . In the course of training for the experiments, mice were handled daily to monitor signs of fighting and body weight, and to help identify the provokers and recipients of aggression. Frequent handling proved critically important to condition and calm mice to some of the tests. For instance, the \"air righting\" test used to evaluate vestibular function involved holding and rotating individual mice cupped in the palms of the experimenter's hands prior to releasing the mouse into a safe fall. Extensive handling and homecage enrichments yielded male mice that were tractable and cooperative during such potentially provocative procedures. Other, useful and informative observations could be made after a change of bedding. Mice become extremely active when placed in a fresh cage. We have repeatedly observed that in unstable groups mice would fight promptly after a change of bedding. In summary, fighting episodes observed after handling or cage cleaning, along with an absence of a communal nest, were treated as indication that the group had to be either rearranged or split and excluded from the experiments.\n",
      "\n",
      "### Food and water consumption\n",
      "\n",
      "The average pelleted food consumption in SFV and GCV groups was 1.300.14 g/10 g bodyweight (BW) and their water consumption was 1.520.65 g/10 g BW (msd). Average paste food consumption in the Space Flight and the Ground Control groups was 5.520.88 g/10 g BW. Taking into account the 76-78%, water content of paste food, dry weight consumed was the same as that for pelleted food of 1.270.20 g/10 g BW (p=0.4514, Mann Whitney test). Notably, water intake via the paste food diet was 4.250.67 g/10 g BW, or roughly three times *ad libitum* water consumption, when water was presented separately from dry, pelleted diet (p&lt;0.0001, Mann Whitney test). Apparently the need for nutrients, rather than thirst, governed the paste food consumption.\n",
      "\n",
      "While no adverse impact of this increased water intake was evident, mice maintained on paste diet produced more urine that was more dilute than normal. When paste food was introduced, water consumption decreased roughly 5 times its original amount; body weight increased by 1.50.4 g, and feces weight tripled ( [Figure 3](#pone-0104830-g003) ). Measured urine production in mice fed pasted food diet was 2.4 ml/10 g BW, which was 10 times the value in mice fed standard diet (Student's paired t=12.55, df=3, p=0.0011).\n",
      "\n",
      "Body weight (A), food (B) and water (C) consumption, feces (D) and urine (E) production. Differences significant at p&lt;0.05 are marked with an asterisk. When the paste food diet with high water content was introduced mice displayed an increase in bodyweight, stopped drinking, while diuresis and feces weight were increased indicating excretion of excess water consumed with the paste diet.\n",
      "\n",
      "<!-- image -->\n",
      "\n",
      "### Telemetry probes impact on the mice welfare\n",
      "\n",
      "Continuous blood pressure measurement, with implantable telemetry probes, was an integral part of the *in vivo* studies. The health status and functional state of the implanted mice were carefully monitored after surgery because the novelty of this procedure as part of a long spaceflight constituted a major risk factor for the mission.\n",
      "\n",
      "Thirty-five (35) mice were implanted with PA-C10 blood pressure transmitters. The mice tolerated with apparent ease telemetry probe implantation. As can be seen from body weight data, the acute recovery was complete by day 5 post-surgery ( [Figure 4](#pone-0104830-g004) ). At approximately the same time, mice were behaviorally recovered in terms of locomotor activity and nest building, and were reunited with their co-habitants.\n",
      "\n",
      "Differences significant at p&lt;0.05 are marked with an asterisk. As can be considered from bodyweight data, acute recovery was over by day 5 after surgery.\n",
      "\n",
      "<!-- image -->\n",
      "\n",
      "Tests of corneal, pineal and other simple reflexes, performance on RotaRod, grip strength, running wheel performance, and learning and memory tests revealed no adverse effects of probe implantation. We occasionally noted, slight eyelid drooping. Although this ptosis could be interpreted as indicating compromised brain blood supply, we do not believe this was the case. Indeed, the circle of Willis is incompletely developed in mice, with 10% incidence in the C57 strain [[32]](#pone.0104830-Ward1) ; the right common carotid artery was ligated during the surgery. The absence of impaired motor or cognitive functions in implanted mice contradicts interpretations of brain blood supply deficiency. An extracranial blood supply through the external carotid could, however, be compromised in operated mice, particularly in the most distal branches of the artery that supply blood to the eyelid. We surmise that was the reason for the slight ptosis observed on the side ipsilateral to the carotid ligation.\n",
      "\n",
      "Mice that underwent surgery were housed individually during acute recovery period (5 days on average) and were re-introduced to their co-habitants when recovered. Two modes of reunion were tested. One method was to introduce two un-operated mice to the cage of an operated animal. This was designed to exploit \"host-intruder\" paradigm and offer an advantage to the host (operated = \"weak\" mouse) due to concerns for its safety. In nearly all cases, however, the non-operated \"intruders\" were the ones to suffer, when the implanted mouse assumed the dominant position. The second method was to introduce all three mice to a fresh cage; this was found to be safe for both operated and intact mice. Based on *in vivo* tests conducted before the experiment, we selected the latter mode of reunion. To summarize, when partially recovered from the operation, mice with implanted probes can be easily group-housed with intact mice and display little (if any) differences from mice without surgery.\n",
      "\n",
      "### Pre-flight training of mice designated for in vivo studies\n",
      "\n",
      "Preliminary training and tests were performed with 42 mice for both the flight (SF) experiment and the corresponding vivarium control (SFV), and with 30 mice for the ground control (GC and GCV groups) experiment ( [Table 3](#pone-0104830-t003) ); 26 (62%) and 16 (53%) mice correspondingly fulfilled the inclusion criteria. Here, it is important to mention that probabilities of fulfilling each of the criteria are below 1. The probabilities of mice displaying a good BP signal, running vigorously in the wheel, performing well in discriminative learning, and maintaining amicable interactions with cagemates should be treated as independent and therefore should be multiplied to get the joint probability. Although the rule is common knowledge, it is easy to overlook when planning an experiment, and anticipating sufficient allowance of extra animals.\n",
      "\n",
      "#### Table 3. The outcome of the pre-flight procedures with mice for in vivo studies.\n",
      "\n",
      "| Criterion                                                                 | Flight + Flight vivarium   | Flight + Flight vivarium   | Flight + Flight vivarium   | Ground control + Ground control vivarium   | Ground control + Ground control vivarium   | Ground control + Ground control vivarium   |\n",
      "|---------------------------------------------------------------------------|----------------------------|----------------------------|----------------------------|--------------------------------------------|--------------------------------------------|--------------------------------------------|\n",
      "|                                                                           | fulfilled, n               | failed, n                  | fulfilled, %               | fulfilled, n                               | failed, n                                  | fulfilled, %                               |\n",
      "| Blood pressure signal quality                                             | 15                         | 5                          | 75                         | 12                                         | 3                                          | 80                                         |\n",
      "| Voluntary running more than 3 km/day                                      | 34                         | 8                          | 81                         | 25                                         | 5                                          | 83                                         |\n",
      "| More than 95% correct reactions in the operant conditioning task by day 3 | 39                         | 3                          | 93                         | 28                                         | 2                                          | 93                                         |\n",
      "| Housing group stability (cages)                                           | 19                         | 2                          | 90                         | 14                                         | 1                                          | 93                                         |\n",
      "\n",
      "[Open in a new tab](table\\pone-0104830-t003)\n",
      "\n",
      "### Transportation\n",
      "\n",
      "The flight group of mice was transported to Cosmodrome Baikonur one week before launch. The relatively long (24 h) transportation to Baikonur did not seriously affect the mice as evidenced by a slight bodyweight change of 3% (0.9 g, [Figure 5](#pone-0104830-g005) ). Bodyweight returned back to pre-transportation values in two days.\n",
      "\n",
      "Differences significant at p&lt;0.05 are marked with an asterisk. Transportation did not seriously affect the mice, as can be concluded from a slight drop of bodyweight and its rapid recovery.\n",
      "\n",
      "<!-- image -->\n",
      "\n",
      "## General Outcomes\n",
      "\n",
      "### Climate\n",
      "\n",
      "The environmental parameters in the biosatellite during the orbital mission, in the climatic chamber during the ground control experiment, and in the animal facility for the VC groups ( [Table 4](#pone-0104830-t004) ), were closely matched and within the acceptable range for the housing of rodents [[33]](#pone.0104830-National1) , [[34]](#pone.0104830-FELASA1) . The climate parameters aboard the satellite were within the designed range throughout the flight [[35]](#pone.0104830-Available1) .\n",
      "\n",
      "#### Table 4. Climate parameters in flight, control experiments and the animal facility.\n",
      "\n",
      "| Parameter              |         | Animal facility   | Biosatellite   | Control experiment   |\n",
      "|------------------------|---------|-------------------|----------------|----------------------|\n",
      "| Temperature, C        | msd    | 23.01.5          | 21.10.4       | 21.30.8             |\n",
      "|                        | min-max | 21.027.0         | 19.522.9      | 20.322.3            |\n",
      "| Relative humidity, %   | msd    | 4917             | 613           | 712                 |\n",
      "|                        | min-max | 2075             | 5365          | 6873                |\n",
      "| Ambient pressure, mmHg | msd    | na                | 7626          | 7473                |\n",
      "|                        | min-max | na                | 752770        | 742753              |\n",
      "| *2*  *O , %*           | msd    | na                | 21.21.1       | 21.20.5             |\n",
      "|                        | min-max | na                | 19.321.2      | 20.621.2            |\n",
      "| *2*  *CO , %*          | msd    | na                | 0.0020.005    | 0.06820.0718        |\n",
      "|                        | min-max | na                | 0.0000.015    | 0.02010.2096        |\n",
      "\n",
      "[Open in a new tab](table\\pone-0104830-t004)\n",
      "\n",
      "na - data not available.\n",
      "\n",
      "### Mice condition after the flight and control experiments\n",
      "\n",
      "In the flight (SF) experiment 16 out of 45 mice (36%) survived and in the ground control (GC) experiment, 38 out of 45 mice (84%) survived. It should be noted that loss of all mice (n=15) in one assembly during space flight was a result of a gross malfunction of the food distribution system. Taking into account that the flight was aimed at exposing mice in the microgravity at, rather than evaluating the long established habitat system, survivorship would be calculated as 16 out of 30 mice (53%). Loss of mice in the SF group was obviously higher than in GC (p&lt;0.0001, Fisher's exact test); however, the results of the control experiment are in concordance with data obtained during preliminary evaluation of the system, where none of the 15 mice died during a 30-day housing in the habitats (p=0.1760, Fisher's exact test). Though the comparison of mice survival in our 30-day automated mission with the data obtained in the 91-day MDS experiment aboard the ISS, where astronauts attended to the mice and survival rate was 50% (3 out of 6), might be dubious, numerically the values do coincide. Upon examination of the surviving SF mice, 4 (25%) had limb injuries and 6 (38%) had various tail injuries. These results were quite unexpected taking into account that, during the preliminary evaluation of the system, all the mice remained in perfect health after 30 days of housing in the habitats.\n",
      "\n",
      "### Mice behavior during the flight\n",
      "\n",
      "To better understand and interpret the fatalities and injuries sustained by some of the mice in the two functioning habitat assemblies, we made use of the uniquely rich video data that were recorded from the Bion habitats (see description, above). It should be noted that never before have such extensive video records been collected and that this design feature alone could earn the \"modernization\" tag in the Bion-M1 designation.\n",
      "\n",
      "The library of video recordings from the two habitat assemblies consisted of 2,476, 30-min segments of individual habitats, captured during both light and dark phases of each day of the mission, as described earlier. As might be expected, video quality was superior in the early phases of the flight and visibility was compromised over time, as floating liquids and debris tended to obscure the camera lens and the general atmosphere of the habitats. More specifically, at the beginning of first week of space flight we categorized as clear with maximal visibility 76% of the video segments. There was a precipitous decrease in visibility at the beginning of the ensuing weeks: the proportion of such clear segments was only 3% and 6% for the 2 nd and 3 rd weeks, respectively. On the first day of the final week none (0%) of the habitats were clearly visible in the video records.\n",
      "\n",
      "Two fatalities occurred in different habitats on the 2 nd day of the flight. In another cage, one mouse died on the 9 th day. There was no evidence of fighting, biting, or agonistic behavior in any of the video samples, so we do not believe that any of deaths or injuries resulted from aggression among the males, evidencing success of our effort to train and select stable housing groups. The first two losses occurred early in the flight, so lack of access to food cannot account for them; and again, in no habitat was there visual evidence of competition that prevented access to food.\n",
      "\n",
      "What might help explain the deaths? The video records clearly showed that the mice spent much time in physical contact (huddling) with one another and, significantly we think, they were grouped against the flat, feeder wall of the cylindrical habitat ( [Figure 6](#pone-0104830-g006) ). We quantified their location within each habitat on select days during the flight; choosing days on which the video segments provided good available visibility. Because we were interested in the aggregative behavior of the mice in different areas, we encoded when there were three, two or one individuals in contact with the feeder wall, the blank wall, or were in the middle area of the habitat. When one mouse was in contact with a flat wall (feeder or blank) and one or more mice were in contact with it but not necessarily with the wall itself, we scored such aggregation as being in wall contact. Using these criteria we found that during the 1 st day of flight, two or three mice were aggregated at the feeder wall for 80% (median value; range: 63%-100%) of the observable segments. They spent 0% of the time at the blank wall. On the 3 rd flight day, aggregations of two or three mice were at the feeder wall for 50% (median; range: 5%-95%) of the observation periods and at the blank wall for a median of 0% (range: 0%-11%). On Day 11, feeder wall aggregations with two or more mice occurred for a median of 46% of the observation time and there were no multiple animal aggregations at the blank wall.\n",
      "\n",
      "Note that mice occupy the floor grid before launch (upper row) and cling to the feeder (lower row) in microgravity. The same cages are shown.\n",
      "\n",
      "<!-- image -->\n",
      "\n",
      "For comparison, we applied the same criteria to the Ground Control habitats for the same days during the simulated flight. Among the GC habitats on day 1, aggregations of two or three mice were seen for a median of 61% of the time (range: 46%-75%) and such aggregations were at the blank wall for only 1.5% of the time (range 0%-4%). On the third and 11 th days the median time aggregations of two or three mice were at the feeder wall was 79% and 95.5%, respectively; there were no such aggregations at the blank wall.\n",
      "\n",
      "In all, these video samples documented from the first day of experimentation, both in microgravity and on earth, that male mice aggregated non-randomly at the feeder wall of the cylindrical habitat. Individuals were often seen oriented toward the food area, which was recessed 1 cm. Whether the behavioral orientation reflected active feeding of the paste diet, or thigmotactic responses to the recess, or grasping of an edge created by the recess or presented by the rotating flap, or some combination of such factors was not discernible from the videos. It was clear, however, that both under 1-g and microgravity conditions, the animals spent most of their aggregative time, which was considerable, near or in contact with the feeder wall. In microgravity, the animals assumed a greater range of orientations at the wall and may have more actively inserted their bodies into the recesses or crevasses created by the hardware, especially the moving parts. This remains supposition, but appears likely and should be considered in future designs as well as in the arrangements for video surveillance during future flights.\n",
      "\n",
      "Results obtained in the Bion-M 1 mission should be taken into account in design of new mice habitats for future space missions. Despite the general design has long since proven its efficiency in the previous Bion flights with rats, including use of paste food and the system of its distribution, further development is needed to better adapt it for mice based on this study; obviously any moving parts in direct access of the animals should be excluded. The advantage of the habitats design is its ability to support mice without any human interference in relatively long duration missions. This degree of autonomy has not been reached by any other of the existing animal habitats, for instance, in the recent MDS mission the water tank had to be refilled every 10 days, and while food supply was designed to operate automatically for 60 days, eventually the food bars had to be manually advanced as needed by the astronauts [[10]](#pone.0104830-Cancedda1) .\n",
      "\n",
      "### Examination at the landing site\n",
      "\n",
      "Examination of mice after the Bion-M 1 flight directly at the landing site (return +3 h) revealed gross motor function impairment: the mice could not maintain steady posture and their fore and hind paws were positioned more to their sides, rather than directly under their trunk (14 of 16, 88%); the mice did not move even when prodded; when lifted, the mice spread their toes excessively (16/16, 100%) and righting reaction was absent or impaired in an aerial righting test (6 of 6, 100%). Some recovery of locomotor function was evident by about 6-8 h after landing, when the mice could keep normal posture and even rear. All mice were found to be exophthalmic. Excessive urinations with almost colorless urine were noted in 14 out of 16 mice.\n",
      "\n",
      "### Body weight data\n",
      "\n",
      "The nutritional status of mice post-flight was variable. Whereas the majority of mice either gained some weight or maintained their pre-flight values (12/16, 75%), 1 mouse was famished (BW 50% below pre-flight value), and 3 were distinctly overweight by 30-40%. The differences in bodyweight between spaceflight, ground control and vivarium control groups were not significant (F (3,104)=1.40, p=0.2468) and they accounted for only 3% of total variation ( [Figure 7](#pone-0104830-g007) ). Post-hoc comparisons revealed no differences between the groups before the experiments. The comparison of pre- versus post-flight values revealed that bodyweight increased in the flight group (8%, p=0.0010), ground control (4%, p=0.0456) and control vivarium groups (11%, p&lt;0.0001) with overall effect of \"time\" being highly significant (F(1,104)=59.79, p&lt;0.0001). The relations of main experimental groups (flight and control experiment groups) to the corresponding vivarium controls were mirrored: while the flight group had gained, on average, more than the mice housed in the animal facility, mice from the control experiment group gained less weight than their vivarium control counterparts. We consider that this discrepancy is explained by little opportunity for exercise in the relatively small habitats in microgravity, and partial loss of mice during the flight experiment, leading to excesses of food available to the survivors. In the control experiment, where the effort was made to reproduce the feeding regimen observed in the flight experiment, given that more mice survived, the relative amount of food per mouse was smaller leading to smaller weight gain. Seasonal variations in growth cannot be ruled out either.\n",
      "\n",
      "The pairs of bars on the (B) panel represent body weight before (left box) and after (right box) the corresponding experiment. Points represent individual mice data, boxes - lower to upper quartile, whiskers - minimum and maximum. Differences significant at p&lt;0.05 are marked with an asterisk.\n",
      "\n",
      "<!-- image -->\n",
      "\n",
      "### Behavior in the open-field test\n",
      "\n",
      "Behavioral activity of individual mice in the open field was recorded during the training period prior to the start of the experiments and one day after landing (SF and SFV), and at corresponding times for the control groups (GC and GCV); the software for behavioral analyses extracted the nine parameters in [Table 5](#pone-0104830-t005) . After return from 30-day spaceflight mice were markedly less active, with both distance ( [Figure 8A, C](#pone-0104830-g008) ) and rearing frequency ( [Figure 8D](#pone-0104830-g008) ) drastically reduced compared to either GC or SFV animals, whereas grooming duration ( [Figure 8H](#pone-0104830-g008) ) was much higher in SF mice compared to any of the control groups. Interestingly, both SF and GC mice who were housed in relatively small habitats displayed more pronounced thigmotaxis compared to mice kept in standard cages (SFV or GCV correspondingly), as evidenced by a decrease of center zone entries ( [Figure 8E](#pone-0104830-g008) ), time spent in center ( [Figure 8F](#pone-0104830-g008) ) and distance moved in center zone, while latency to the first entry into the center zone was increased in SF mice only ( [Figure 8G](#pone-0104830-g008) ). Behavior parameters of mice from two vivarium groups, SFV and GCV, were indistinguishable.\n",
      "\n",
      "#### Table 5. Open field test.\n",
      "\n",
      "| Parameter                               | Parameter   | SF (n=5)   | SFV (n=17)   | GC (n=9)   | GCV (n=12)   |\n",
      "|-----------------------------------------|-------------|--------------|----------------|--------------|----------------|\n",
      "| Distance moved, m                       | Pre         | 83.96.2     | 84.26.3       | 82.84.5     | 85.75.9       |\n",
      "|                                         | Post        | 39.05.5     | 69.73.4       | 63.24.6     | 69.14.3       |\n",
      "| Distance moved in center zone, m        | Pre         | 31.23.8     | 34.13.2       | 32.23.5     | 30.42.8       |\n",
      "|                                         | Post        | 10.42.1     | 31.91.7       | 22.12.9     | 29.03.1       |\n",
      "| Distance moved in the periphery zone, m | Pre         | 52.75.2     | 50.13.7       | 50.62.0     | 55.33.7       |\n",
      "|                                         | Post        | 28.65.1     | 37.82.5       | 41.12.8     | 40.12.4       |\n",
      "| Movement speed, m/min                   | Pre         | 5.500.56    | 5.010.22      | 4.850.26    | 5.270.36      |\n",
      "|                                         | Post        | 3.450.49    | 4.210.23      | 3.840.23    | 4.230.21      |\n",
      "| Rearing frequency                       | Pre         | 17014       | 16912         | 14710       | 13211         |\n",
      "|                                         | Post        | 5311        | 1668          | 1387        | 12814         |\n",
      "| Center zone entry frequency             | Pre         | 8310        | 918           | 818         | 806           |\n",
      "|                                         | Post        | 306         | 785           | 565         | 756           |\n",
      "| Time in center zone, s                  | Pre         | 27950       | 36742         | 36722       | 31023         |\n",
      "|                                         | Post        | 16155       | 43430         | 33846       | 42340         |\n",
      "| Latency to first center zone entry, s   | Pre         | 136         | 112           | 2810        | 205           |\n",
      "|                                         | Post        | 5213        | 214           | 3711        | 279           |\n",
      "| Grooming duration, s                    | Pre         | 8331        | 8814          | 726         | 896           |\n",
      "|                                         | Post        | 31952       | 11413         | 8814        | 10327         |\n",
      "\n",
      "[Open in a new tab](table\\pone-0104830-t005)\n",
      "\n",
      "Total (A), center (B) and periphery (C) distance moved, rearing frequency (D), center entries frequency (E), time in center (F), latency to the first center zone entry (G) and grooming duration (H). SF mice displayed reduced activity compared to any of the control groups (GC, SFV or GCV). Mice housed in habitats (SF and GC) were reluctant to explore the center of the arena. Statistical analysis was performed using Mann-Whitney test (*p&lt;0.05, **p&lt;0.01, *** p&lt;0.005 and ****p&lt;0.0001, ns - not significant or the p value is indicated).\n",
      "\n",
      "<!-- image -->\n",
      "\n",
      "The drastic decrease in activity by SF mice in this test is obviously related to adverse impact of prolonged 30-days spaceflight on the sensorimotor systems of these animals. In part, housing within the restricted confines of the habitats decreased the propensity of the mice to enter and explore the center of the arena; such observations are usually interpreted in terms of increased anxiety in these animals compared to cage-housed controls [[20]](#pone.0104830-Gould1) . Compared the flight habitat interior, animals maintained in the animal facility experienced enriched environment and such differences are associated with greater exploration of the central area in the OFT [[36]](#pone.0104830-Lin1) . In contrast, increased housing density, experienced by SF and GC mice, had little effect on the central area exploration. In the present case the higher animal density, sometimes associated with increased central exploration and reduced anxiety [[37]](#pone.0104830-Morgan1) , did not prevail.\n",
      "\n",
      "The example of OFT data illustrates the efficacy of our training program and experimental design for promoting scientific output of the studies performed with mice after landing in the Bion-M 1 program. Because OFT activity is known to decrease with repetitive tests, the repeated testing approach used in this study could potentially obscure the effects of spaceflight. Nevertheless, the opposite was true. While some of the measures taken before the flight, e.g., distance moved, were very close in different experimental groups with variation coefficients ranging between 16 and 30%, others differed between the groups and were quite variable within the group, e.g. latency to enter central zone with within group coefficient of variation of 70-110% ( [Table 5](#pone-0104830-t005) ). In order to reduce this inter-individual and seasonal variability, results of preliminary testing were used to make the correction for baseline differences; this transformation made possible differentiation of spaceflight and habitat effects ( [Figure 8](#pone-0104830-g008) ).\n",
      "\n",
      "### Post-flight adjustment of experimental groups\n",
      "\n",
      "Due to the partial loss of mice during the flight, the number of animals in other groups had to be adjusted ( [Table 6](#pone-0104830-t006) ). The number of control mice (GC, SFV, GCV) for *in vitro* studies was decreased proportionately to the number of SF animals available. This decision was made basing on the relatively small variability of the measures to be taken in inbred C57 mice [[12]](#pone.0104830-Festing1) and time considerations (complete dissection of one animal lasted 1.0-1.2 hours and dissection of all the mice took more than 24 hours). Taking that accidental death of mice in flight was a random event and that bodyweight data from all the animals was available, the mice for control groups were selected to match the initial bodyweight of SF mice. Different tactics were chosen for the *in vivo* subgroups. The behavioral data were characterized by high variability, at the same time all the animals designated for *in vivo* studies underwent preliminary testing and selection, taking this into account the number of mice for *in vivo* studies was not altered in order to obtain a better estimate of the behavioral parameters in the batch of mice used and to increase the power of the planned analyses.\n",
      "\n",
      "#### Table 6. Number of animals in experimental groups after post-flight adjustment.\n",
      "\n",
      "| Group                                              |   In vitro studies |   In vivo studies |\n",
      "|----------------------------------------------------|--------------------|-------------------|\n",
      "| Flight experiment (SF)                             |                 11 |                 5 |\n",
      "| Flight vivarium control (SFV)                      |                 18 |                20 |\n",
      "| Ground control experiment (GC)                     |                 16 |                 9 |\n",
      "| Ground control experiment  vivarium control (GCV) |                 16 |                13 |\n",
      "\n",
      "[Open in a new tab](table\\pone-0104830-t006)\n",
      "\n",
      "### Implementation of the scientific program\n",
      "\n",
      "Despite significant loss of animals in flight, the scientific program was not abridged and was implemented in full. Hemodynamic and behavioral data were collected throughout the flight. Dissection of SF and SFV mice started 12 hours after landing on May 20 th , 2013. Recovery was followed for 7 days in the *in vivo* subgroup of SF mice and the dissection of these animals was performed on May 26. Joint effort of teams from Russia, USA, France, Germany, Italy and Ukraine who took part in the *in vivo* studies and tissue-sharing program resulted in over 70 distinct hypothesis-driven studies.\n",
      "\n",
      "## Conclusions\n",
      "\n",
      "The Bion-M 1 flight was unique in many ways, particularly the unprecedented duration of an automated mission with living animals. The sex of the mice was male, thus imposing greater attention to creation and selection of stable groups of mice for social housing. The overall success was evident: aggressive interactions were noted on single occasions and under conditions of starvation only. The use of male animals is obviously required for research in male reproduction and their larger size compared to females offered clear advantages. Our program of mice training for flight and control experiments of the Bion-M 1 project was aimed to: ensure that mice adapt to stressful conditions of space flight when housed in groups; to collect baseline data from these mice; select homogenous groups; and, more generally, to promote collection of less variable data. The results of our program presented here demonstrate that group housed male mice need not be excluded *a priori* , but quite on the contrary, can be successfully employed in space biomedical research if clear and simple procedures provided by this study are followed.\n",
      "\n",
      "The failures in this project were connected with hardware malfunction in food distribution system and to microgravity-related changes in mouse behavior. Some of the injuries and deaths encountered in this pioneering effort suggest moving parts within the habitat present potential risks and should be avoided or carefully analyzed in the design of future mouse habitats.\n",
      "\n",
      "In this space experiment we have, for the first time, introduced implantable telemetry in these small laboratory animals. This shift to data collection during the flight became possible with the development of miniature implantable probes, thereby dramatically increasing the scientific output of such experiments. Moreover, the use of an automated data collection system during the post-flight stage of this experiment provided comprehensive data on the course and mechanism of recovery of varied physiological systems following exposure to microgravity. The unprecedented inclusion of daily video recordings proved valuable for scientific purposes, for animal welfare considerations, and for assessing both nominal and aberrant events.\n",
      "\n",
      "The good condition of spaceflight mice was the basis for the positive results of both the recovery and tissue sharing studies; leading us to conclude that our training program was generally effective. We express our hope that the results and considerations presented here will be useful for the planning of future missions and will help to ensure the welfare of rodents in space research.\n",
      "\n",
      "## Acknowledgments\n",
      "\n",
      "The authors thank the staff of MSU Institute of Mitoengineering, and in particular Prof. Vladimir Skulachev, Maxim and Innokentiy Skulachev for their kind and assisting help with the organization of the mice preparation procedures. The authors are indebted to the participants of the MTB project: Guillemette Gauquelin-Koch, Patrick Aubry, Jean-Christophe Lloret and Marc-Antoine Custaud (CNES, France); Anatoliy Belgorodskiy and his colleagues from Biofizpribor (Russia); and Andrey Pakhomov (IMBP), who made possible the telemetric recording of blood pressure in mice for this mission. The authors deeply respect the contribution to the success of this mission made by Evgeniya Yarmanova and Vyacheslav Sedletskiy at the most critical moments. The authors are grateful to the staff of Kurchatov NBIC-center for their involvement in this project. In addition, special thanks to Evgeniya Lagereva for participating enthusiastically in the routine animal procedures, and to Nikolay Buzin for the record-braking speed of the Phenomaster data collection system refurbishment.\n",
      "\n",
      "## Funding Statement\n",
      "\n",
      "The project was supported by the Russian Federal Space Agency and the Russian Academy of Sciences. These funders took part in study design and data collection, but had no role in data analysis or decision to publish. The study was also supported by the Division of Physiology and Fundamental Medicine of Russian Academy of Sciences (\"Integrative physiology\" program) and RFBR grant 12-04-01665-a. These funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.\n",
      "\n",
      "## References\n",
      "\n",
      "- 1. Grigoriev AI, Iliyn EA (2007) Animals in space. Vestnik Rossijskoj Akademii Nauk 77: 963-973. [ [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Vestnik%20Rossijskoj%20Akademii%20Nauk&title=Animals%20in%20space&author=AI%20Grigoriev&author=EA%20Iliyn&volume=77&publication_year=2007&pages=963-973&) ]\n",
      "- 2. Ilyin EA (2000) Historical overview of the Bion project. J Gravit Physiol 7: S1-S8. [ [PubMed](https://pubmed.ncbi.nlm.nih.gov/11543436/) ] [ [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Gravit%20Physiol&title=Historical%20overview%20of%20the%20Bion%20project&author=EA%20Ilyin&volume=7&publication_year=2000&pages=S1-S8&pmid=11543436&) ]\n",
      "- 3. Burgess C, Dubbs C (2007) Animals in Space. From Research Rockets to the Space Shuttle. UK: Springer Praxis. 406 p. [ [Google Scholar](https://scholar.google.com/scholar_lookup?Burgess%20C,%20Dubbs%20C%20(2007)%20Animals%20in%20Space.%20From%20Research%20Rockets%20to%20the%20Space%20Shuttle.%20UK:%20Springer%20Praxis.%20406%20p.) ]\n",
      "- 4. Buckey JC, Homick JL, editors (2003) The Neurolab Spacelab mission: neuroscience research in space: results from the STS-90, Neurolab Spacelab mission. Houston: National Aeronautics and Space Administration, Lyndon B. Johnson Space Center. 333 p. [ [Google Scholar](https://scholar.google.com/scholar_lookup?Buckey%20JC,%20Homick%20JL,%20editors%20(2003)%20The%20Neurolab%20Spacelab%20mission:%20neuroscience%20research%20in%20space:%20results%20from%20the%20STS-90,%20Neurolab%20Spacelab%20mission.%20Houston:%20National%20Aeronautics%20and%20Space%20Administration,%20Lyndon%20B.%20Johnson%20Space%20Center.%20333%20p.) ]\n",
      "- 5. Pecaut MJ, Nelson GA, Peters LL, Kostenuik PJ, Bateman TA, et al. (2003) Genetic models in applied physiology: selected contribution: effects of spaceflight on immunity in the C57BL/6 mouse. I. Immune population distributions. J Appl Physiol 94: 2085-2094. [ [DOI](https://doi.org/10.1152/japplphysiol.01052.2002) ] [ [PubMed](https://pubmed.ncbi.nlm.nih.gov/12514166/) ] [ [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Appl%20Physiol&title=Genetic%20models%20in%20applied%20physiology:%20selected%20contribution:%20effects%20of%20spaceflight%20on%20immunity%20in%20the%20C57BL/6%20mouse.%20I.%20Immune%20population%20distributions&author=MJ%20Pecaut&author=GA%20Nelson&author=LL%20Peters&author=PJ%20Kostenuik&author=TA%20Bateman&volume=94&publication_year=2003&pages=2085-2094&pmid=12514166&doi=10.1152/japplphysiol.01052.2002&) ]\n",
      "- 6. Behnke BJ, Stabley JN, McCullough DJ, Davis RT, Dominguez JM, et al. (2013) Effects of spaceflight and ground recovery on mesenteric artery and vein constrictor properties in mice. FASEB J 27: 399-409. [ [DOI](https://doi.org/10.1096/fj.12-218503) ] [ [PMC free article](\\articles\\PMC4050424) ] [ [PubMed](https://pubmed.ncbi.nlm.nih.gov/23099650/) ] [ [Google Scholar](https://scholar.google.com/scholar_lookup?journal=FASEB%20J&title=Effects%20of%20spaceflight%20and%20ground%20recovery%20on%20mesenteric%20artery%20and%20vein%20constrictor%20properties%20in%20mice&author=BJ%20Behnke&author=JN%20Stabley&author=DJ%20McCullough&author=RT%20Davis&author=JM%20Dominguez&volume=27&publication_year=2013&pages=399-409&pmid=23099650&doi=10.1096/fj.12-218503&) ]\n",
      "- 7. Gridley DS, Mao XW, Stodieck LS, Ferguson VL, Bateman TA, et al. (2013) Changes in Mouse Thymus and Spleen after Return from the STS-135 Mission in Space. PLoS ONE 8: e75097. [ [DOI](https://doi.org/10.1371/journal.pone.0075097) ] [ [PMC free article](\\articles\\PMC3777930) ] [ [PubMed](https://pubmed.ncbi.nlm.nih.gov/24069384/) ] [ [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20ONE&title=Changes%20in%20Mouse%20Thymus%20and%20Spleen%20after%20Return%20from%20the%20STS-135%20Mission%20in%20Space&author=DS%20Gridley&author=XW%20Mao&author=LS%20Stodieck&author=VL%20Ferguson&author=TA%20Bateman&volume=8&publication_year=2013&pages=e75097&pmid=24069384&doi=10.1371/journal.pone.0075097&) ]\n",
      "- 8. Baqai FP, Gridley DS, Slater JM, Luo-Owen X, Stodieck LS, et al. (2009) Effects of spaceflight on innate immune function and antioxidant gene expression. J Appl Physiol 106: 1935-1942. [ [DOI](https://doi.org/10.1152/japplphysiol.91361.2008) ] [ [PMC free article](\\articles\\PMC2692779) ] [ [PubMed](https://pubmed.ncbi.nlm.nih.gov/19342437/) ] [ [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Appl%20Physiol&title=Effects%20of%20spaceflight%20on%20innate%20immune%20function%20and%20antioxidant%20gene%20expression&author=FP%20Baqai&author=DS%20Gridley&author=JM%20Slater&author=X%20Luo-Owen&author=LS%20Stodieck&volume=106&publication_year=2009&pages=1935-1942&pmid=19342437&doi=10.1152/japplphysiol.91361.2008&) ]\n",
      "- 9. Lujan HL, Janbaih H, Feng HZ, Jin JP, DiCarlo SE (2006) Reduced susceptibility to ventricular tachyarrhythmias in rats selectively bred for high aerobic capacity. AJP: Heart and Circulatory Physiology 291: H2933-H2941. [ [DOI](https://doi.org/10.1152/ajpheart.00514.2006) ] [ [PubMed](https://pubmed.ncbi.nlm.nih.gov/16891405/) ] [ [Google Scholar](https://scholar.google.com/scholar_lookup?journal=AJP:%20Heart%20and%20Circulatory%20Physiology&title=Reduced%20susceptibility%20to%20ventricular%20tachyarrhythmias%20in%20rats%20selectively%20bred%20for%20high%20aerobic%20capacity&author=HL%20Lujan&author=H%20Janbaih&author=HZ%20Feng&author=JP%20Jin&author=SE%20DiCarlo&volume=291&publication_year=2006&pages=H2933-H2941&pmid=16891405&doi=10.1152/ajpheart.00514.2006&) ]\n",
      "- 10. Cancedda R, Liu Y, Ruggiu A, Tavella S, Biticchi R, et al. (2012) The Mice Drawer System (MDS) Experiment and the Space Endurance Record-Breaking Mice. PLoS ONE 7: e32243. [ [DOI](https://doi.org/10.1371/journal.pone.0032243) ] [ [PMC free article](\\articles\\PMC3362598) ] [ [PubMed](https://pubmed.ncbi.nlm.nih.gov/22666312/) ] [ [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20ONE&title=The%20Mice%20Drawer%20System%20(MDS)%20Experiment%20and%20the%20Space%20Endurance%20Record-Breaking%20Mice&author=R%20Cancedda&author=Y%20Liu&author=A%20Ruggiu&author=S%20Tavella&author=R%20Biticchi&volume=7&publication_year=2012&pages=e32243&pmid=22666312&doi=10.1371/journal.pone.0032243&) ]\n",
      "- 11. Andreev-Andrievskiy AA, Popova AS, Borovik AS, Dolgov ON, Tsvirkun DV, et al. (2014) Physiology &amp; Behavior. 132: 1-9. [ [DOI](https://doi.org/10.1016/j.physbeh.2014.03.033) ] [ [PubMed](https://pubmed.ncbi.nlm.nih.gov/24802359/) ] [ [Google Scholar](https://scholar.google.com/scholar_lookup?Andreev-Andrievskiy%20AA,%20Popova%20AS,%20Borovik%20AS,%20Dolgov%20ON,%20Tsvirkun%20DV,%20et%20al.%20(2014)%20Physiology%20&%20Behavior.%20132:%201%E2%80%939.) ]\n",
      "- 12. Festing MF, Overend P, Gaines Das R, Cortina-Borja M, Berdoy M (2002) The design of animal experiments: reducing the use of animals in research through better experimental design. London: Royal Society of Medicine. 120 p. [ [Google Scholar](https://scholar.google.com/scholar_lookup?Festing%20MF,%20Overend%20P,%20Gaines%20Das%20R,%20Cortina-Borja%20M,%20Berdoy%20M%20(2002)%20The%20design%20of%20animal%20experiments:%20reducing%20the%20use%20of%20animals%20in%20research%20through%20better%20experimental%20design.%20London:%20Royal%20Society%20of%20Medicine.%20120%20p.) ]\n",
      "- 13. European Convention for the Protection of Vertebrate Animals Used for Experimental and Other Scientific Purposes: Convention Europenne Sur la Protection Des Animaux Vertbrs Utiliss  Des Fins Exprimentales Ou  D'autres Fins Scientifiques:[Strasbourg, 18. III. 1986]\n",
      "- 14. Aviles H, Belay T, Fountain K, Vance M, Sonnenfeld G (2003) Increased susceptibility to Pseudomonas aeruginosa infection under hindlimb-unloading conditions. J Appl Physiol 95: 73-80. [ [DOI](https://doi.org/10.1152/japplphysiol.00968.2002) ] [ [PubMed](https://pubmed.ncbi.nlm.nih.gov/12626488/) ] [ [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Appl%20Physiol&title=Increased%20susceptibility%20to%20Pseudomonas%20aeruginosa%20infection%20under%20hindlimb-unloading%20conditions&author=H%20Aviles&author=T%20Belay&author=K%20Fountain&author=M%20Vance&author=G%20Sonnenfeld&volume=95&publication_year=2003&pages=73-80&pmid=12626488&doi=10.1152/japplphysiol.00968.2002&) ]\n",
      "- 15. Wilson JW, Ott CM, Quick L, Davis R, zu Bentrup KH, et al. (2008) Media Ion Composition Controls Regulatory and Virulence Response of Salmonella in Spaceflight. PLoS ONE 3: e3923. [ [DOI](https://doi.org/10.1371/journal.pone.0003923) ] [ [PMC free article](\\articles\\PMC2592540) ] [ [PubMed](https://pubmed.ncbi.nlm.nih.gov/19079590/) ] [ [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20ONE&title=Media%20Ion%20Composition%20Controls%20Regulatory%20and%20Virulence%20Response%20of%20Salmonella%20in%20Spaceflight&author=JW%20Wilson&author=CM%20Ott&author=L%20Quick&author=R%20Davis&author=KH%20zu%20Bentrup&volume=3&publication_year=2008&pages=e3923&pmid=19079590&doi=10.1371/journal.pone.0003923&) ]\n",
      "- 16. Howard BR (2002) Control of variability. ILAR J 43: 194-201. [ [DOI](https://doi.org/10.1093/ilar.43.4.194) ] [ [PubMed](https://pubmed.ncbi.nlm.nih.gov/12391394/) ] [ [Google Scholar](https://scholar.google.com/scholar_lookup?journal=ILAR%20J&title=Control%20of%20variability&author=BR%20Howard&volume=43&publication_year=2002&pages=194-201&pmid=12391394&doi=10.1093/ilar.43.4.194&) ]\n",
      "- 17. Van Loo PLP, Van Zutphen LFM, Baumans V (2003) Male management: coping with aggression problems in male laboratory mice. Laboratory Animals 37: 300-313. [ [DOI](https://doi.org/10.1258/002367703322389870) ] [ [PubMed](https://pubmed.ncbi.nlm.nih.gov/14599305/) ] [ [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Laboratory%20Animals&title=Male%20management:%20coping%20with%20aggression%20problems%20in%20male%20laboratory%20mice&author=PLP%20Van%20Loo&author=LFM%20Van%20Zutphen&author=V%20Baumans&volume=37&publication_year=2003&pages=300-313&pmid=14599305&doi=10.1258/002367703322389870&) ]\n",
      "- 18. Kurtz TW (2005) Recommendations for Blood Pressure Measurement in Humans and Experimental Animals: Part 2: Blood Pressure Measurement in Experimental Animals: A Statement for Professionals From the Subcommittee of Professional and Public Education of the American Heart Association Council on High Blood Pressure Research. [ [DOI](https://doi.org/10.1161/01.HYP.0000150857.39919.cb) ] [ [PubMed](https://pubmed.ncbi.nlm.nih.gov/15611363/) ]\n",
      "- 19. Schneider I, Tirsch WS, Faus-Keler T, Becker L, Kling E, et al. (2006) Systematic, standardized and comprehensive neurological phenotyping of inbred mice strains in the German Mouse Clinic. Journal of Neuroscience Methods 157: 82-90. [ [DOI](https://doi.org/10.1016/j.jneumeth.2006.04.002) ] [ [PubMed](https://pubmed.ncbi.nlm.nih.gov/16720049/) ] [ [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Neuroscience%20Methods&title=Systematic,%20standardized%20and%20comprehensive%20neurological%20phenotyping%20of%20inbred%20mice%20strains%20in%20the%20German%20Mouse%20Clinic&author=I%20Schneider&author=WS%20Tirsch&author=T%20Faus-Ke%CE%B2ler&author=L%20Becker&author=E%20Kling&volume=157&publication_year=2006&pages=82-90&pmid=16720049&doi=10.1016/j.jneumeth.2006.04.002&) ]\n",
      "- 20. Gould TD, Dao DT, Kovacsics CE (2009) The Open Field Test. In: T.D. Gould (ed.), Mood and Anxiety Related Phenotypes in Mice, Neuromethods. 42. NewYork: Humana press. pp. 1-20. [ [Google Scholar](https://scholar.google.com/scholar_lookup?Gould%20TD,%20Dao%20DT,%20Kovacsics%20CE%20(2009)%20The%20Open%20Field%20Test.%20In:%20T.D.%20Gould%20(ed.),%20Mood%20and%20Anxiety%20Related%20Phenotypes%20in%20Mice,%20Neuromethods.%2042.%20NewYork:%20Humana%20press.%20pp.%201%E2%80%9320.) ]\n",
      "- 21. Maurissen JPJ, Marable BR, Andrus AK, Stebbins KE (2003) Factors affecting grip strength testing. Neurotoxicology and Teratology 25: 543-553. [ [DOI](https://doi.org/10.1016/s0892-0362(03)00073-4) ] [ [PubMed](https://pubmed.ncbi.nlm.nih.gov/12972067/) ] [ [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Neurotoxicology%20and%20Teratology&title=Factors%20affecting%20grip%20strength%20testing&author=JPJ%20Maurissen&author=BR%20Marable&author=AK%20Andrus&author=KE%20Stebbins&volume=25&publication_year=2003&pages=543-553&pmid=12972067&doi=10.1016/s0892-0362(03)00073-4&) ]\n",
      "- 22. Rogers D, Peters J, Martin J, Ball S, Nicholson S, et al. (2001) SHIRPA, a protocol for behavioral assessment: validation for longitudinal study of neurological dysfunction in mice. Neurosci Lett 306: 89-92. [ [DOI](https://doi.org/10.1016/s0304-3940(01)01885-7) ] [ [PubMed](https://pubmed.ncbi.nlm.nih.gov/11403965/) ] [ [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Neurosci%20Lett&title=SHIRPA,%20a%20protocol%20for%20behavioral%20assessment:%20validation%20for%20longitudinal%20study%20of%20neurological%20dysfunction%20in%20mice&author=D%20Rogers&author=J%20Peters&author=J%20Martin&author=S%20Ball&author=S%20Nicholson&volume=306&publication_year=2001&pages=89-92&pmid=11403965&doi=10.1016/s0304-3940(01)01885-7&) ]\n",
      "- 23. Malkki HAI (2010) Appetitive operant conditioning in mice: heritability and dissociability of training stages. Front Behav Neurosci. 4. Article 171. [ [DOI](https://doi.org/10.3389/fnbeh.2010.00171) ] [ [PMC free article](\\articles\\PMC2990458) ] [ [PubMed](https://pubmed.ncbi.nlm.nih.gov/21119771/) ]\n",
      "- 24. Conover WJ, Iman RL (1981) Rank Transformations as a Bridge Between Parametric and Nonparametric Statistics. The American Statistician 35: 124-129. [ [Google Scholar](https://scholar.google.com/scholar_lookup?journal=The%20American%20Statistician&title=Rank%20Transformations%20as%20a%20Bridge%20Between%20Parametric%20and%20Nonparametric%20Statistics&author=WJ%20Conover&author=RL%20Iman&volume=35&publication_year=1981&pages=124-129&) ]\n",
      "- 25. Newberry RC (1995) Environmental enrichment: increasing the biological relevance of captive environments. Applied Animal Behaviour Science 44: 229-243. [ [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Applied%20Animal%20Behaviour%20Science&title=Environmental%20enrichment:%20increasing%20the%20biological%20relevance%20of%20captive%20environments&author=RC%20Newberry&volume=44&publication_year=1995&pages=229-243&) ]\n",
      "- 26. Van de Weerd HA, Van Loo P, Van Zutphen L, Koolhaas JM, Baumans V (1998) Strength of preference for nesting material as environmental enrichment for laboratory mice. Applied Animal Behaviour Science 55: 369-382. [ [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Applied%20Animal%20Behaviour%20Science&title=Strength%20of%20preference%20for%20nesting%20material%20as%20environmental%20enrichment%20for%20laboratory%20mice&author=HA%20Van%20de%20Weerd&author=P%20Van%20Loo&author=L%20Van%20Zutphen&author=JM%20Koolhaas&author=V%20Baumans&volume=55&publication_year=1998&pages=369-382&) ]\n",
      "- 27. Bouwknecht JA, Hijzen TH, van der Gugten J, Maes RA, Olivier B (2000) Stress-induced hyperthermia in mice: effects of flesinoxan on heart rate and body temperature. Eur J Pharmacol 400: 59-66. [ [DOI](https://doi.org/10.1016/s0014-2999(00)00387-3) ] [ [PubMed](https://pubmed.ncbi.nlm.nih.gov/10913585/) ] [ [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Pharmacol&title=Stress-induced%20hyperthermia%20in%20mice:%20effects%20of%20flesinoxan%20on%20heart%20rate%20and%20body%20temperature&author=JA%20Bouwknecht&author=TH%20Hijzen&author=J%20van%20der%20Gugten&author=RA%20Maes&author=B%20Olivier&volume=400&publication_year=2000&pages=59-66&pmid=10913585&doi=10.1016/s0014-2999(00)00387-3&) ]\n",
      "- 28. van Bogaert MJV, Groenink L, Oosting RS, Westphal KGC, van der Gugten J, et al. (2006) Mouse strain differences in autonomic responses to stress. Genes Brain Behav 5: 139-149. [ [DOI](https://doi.org/10.1111/j.1601-183X.2005.00143.x) ] [ [PubMed](https://pubmed.ncbi.nlm.nih.gov/16507005/) ] [ [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genes%20Brain%20Behav&title=Mouse%20strain%20differences%20in%20autonomic%20responses%20to%20stress&author=MJV%20van%20Bogaert&author=L%20Groenink&author=RS%20Oosting&author=KGC%20Westphal&author=J%20van%20der%20Gugten&volume=5&publication_year=2006&pages=139-149&pmid=16507005&doi=10.1111/j.1601-183X.2005.00143.x&) ]\n",
      "- 29. Drude S, Geiler A, Olfe J, Starke A, Domanska G, et al. (2011) Side effects of control treatment can conceal experimental data when studying stress responses to injection and psychological stress in mice. Lab Animal 40: 119-128. [ [DOI](https://doi.org/10.1038/laban0411-119) ] [ [PubMed](https://pubmed.ncbi.nlm.nih.gov/21427691/) ] [ [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lab%20Animal&title=Side%20effects%20of%20control%20treatment%20can%20conceal%20experimental%20data%20when%20studying%20stress%20responses%20to%20injection%20and%20psychological%20stress%20in%20mice&author=S%20Drude&author=A%20Gei%CE%B2ler&author=J%20Olfe&author=A%20Starke&author=G%20Domanska&volume=40&publication_year=2011&pages=119-128&pmid=21427691&doi=10.1038/laban0411-119&) ]\n",
      "- 30. Garipy J-L, Rodriguiz RM, Jones BC (2002) Handling, genetic and housing effects on the mouse stress system, dopamine function, and behavior. Pharmacol Biochem Behav 73: 7-17. [ [DOI](https://doi.org/10.1016/s0091-3057(02)00789-x) ] [ [PubMed](https://pubmed.ncbi.nlm.nih.gov/12076720/) ] [ [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacol%20Biochem%20Behav&title=Handling,%20genetic%20and%20housing%20effects%20on%20the%20mouse%20stress%20system,%20dopamine%20function,%20and%20behavior&author=J-L%20Gari%C3%A9py&author=RM%20Rodriguiz&author=BC%20Jones&volume=73&publication_year=2002&pages=7-17&pmid=12076720&doi=10.1016/s0091-3057(02)00789-x&) ]\n",
      "- 31. Hurst JL, West RS (2010) Taming anxiety in laboratory mice. Nat Meth 7: 825-826. [ [DOI](https://doi.org/10.1038/nmeth.1500) ] [ [PubMed](https://pubmed.ncbi.nlm.nih.gov/20835246/) ] [ [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Meth&title=Taming%20anxiety%20in%20laboratory%20mice&author=JL%20Hurst&author=RS%20West&volume=7&publication_year=2010&pages=825-826&pmid=20835246&doi=10.1038/nmeth.1500&) ]\n",
      "- 32. Ward R, Collins RL, Tanguay G, Miceli D (1990) A quantitative study of cerebrovascular variation in inbred mice. J Anat 173: 87-95. [ [PMC free article](\\articles\\PMC1256083) ] [ [PubMed](https://pubmed.ncbi.nlm.nih.gov/2074233/) ] [ [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Anat&title=A%20quantitative%20study%20of%20cerebrovascular%20variation%20in%20inbred%20mice&author=R%20Ward&author=RL%20Collins&author=G%20Tanguay&author=D%20Miceli&volume=173&publication_year=1990&pages=87-95&pmid=2074233&) ]\n",
      "- 33. National Research Council (2010) Guide for the Care and Use of Laboratory Animals. Washington: National Academies Press. p 41-103. [ [Google Scholar](https://scholar.google.com/scholar_lookup?National%20Research%20Council%20(2010)%20Guide%20for%20the%20Care%20and%20Use%20of%20Laboratory%20Animals.%20Washington:%20National%20Academies%20Press.%20p%2041%E2%80%93103.) ]\n",
      "- 34. FELASA (2007) Euroguide On the Accommodation and Care of Animals Used for Experimental and Other Scientific Purposes. London: Royal Society of Medicine. 65 pp. [ [Google Scholar](https://scholar.google.com/scholar_lookup?FELASA%20(2007)%20Euroguide%20On%20the%20Accommodation%20and%20Care%20of%20Animals%20Used%20for%20Experimental%20and%20Other%20Scientific%20Purposes.%20London:%20Royal%20Society%20of%20Medicine.%2065%20pp.) ]\n",
      "- 35. Available: [http://biosputnik.imbp.ru](http://biosputnik.imbp.ru/) . Accessed 19 December 2013.\n",
      "- 36. Lin E-JD, Choi E, Liu X, Martin A, During MJ (2011) Environmental enrichment exerts sex-specific effects on emotionality in C57BL/6J mice. Behavioural Brain Research 216: 349-357. [ [DOI](https://doi.org/10.1016/j.bbr.2010.08.019) ] [ [PMC free article](\\articles\\PMC3727905) ] [ [PubMed](https://pubmed.ncbi.nlm.nih.gov/20732356/) ] [ [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Behavioural%20Brain%20Research&title=Environmental%20enrichment%20exerts%20sex-specific%20effects%20on%20emotionality%20in%20C57BL/6J%20mice&author=E-JD%20Lin&author=E%20Choi&author=X%20Liu&author=A%20Martin&author=MJ%20During&volume=216&publication_year=2011&pages=349-357&pmid=20732356&doi=10.1016/j.bbr.2010.08.019&) ]\n",
      "- 37. Morgan JL, Svenson KL, Lake JP, Zhang W, Stearns TM, et al. (2014) Effects of Housing Density in Five Inbred Strains of Mice. PLoS ONE 9: e90012. [ [DOI](https://doi.org/10.1371/journal.pone.0090012) ] [ [PMC free article](\\articles\\PMC3962340) ] [ [PubMed](https://pubmed.ncbi.nlm.nih.gov/24658028/) ] [ [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20ONE&title=Effects%20of%20Housing%20Density%20in%20Five%20Inbred%20Strains%20of%20Mice&author=JL%20Morgan&author=KL%20Svenson&author=JP%20Lake&author=W%20Zhang&author=TM%20Stearns&volume=9&publication_year=2014&pages=e90012&pmid=24658028&doi=10.1371/journal.pone.0090012&) ]\n",
      "\n",
      "## ACTIONS\n",
      "\n",
      "- [View on publisher site](https://doi.org/10.1371/journal.pone.0104830)\n",
      "- [PDF (2.4 MB)](pdf\\pone.0104830.pdf)\n",
      "- Cite\n",
      "- Collections\n",
      "- Permalink\n",
      "\n",
      "## RESOURCES\n",
      "\n",
      "### Similar articles\n",
      "\n",
      "### Cited by other articles\n",
      "\n",
      "### Links to NCBI Databases\n",
      "\n",
      "Back to Top\n"
     ]
    }
   ],
   "execution_count": 9
  },
  {
   "metadata": {},
   "cell_type": "code",
   "outputs": [],
   "execution_count": null,
   "source": "",
   "id": "87eb755d80a5065b"
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 2
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython2",
   "version": "2.7.6"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
